INFO       2024-03-06 17:44:25,716 __main__ <module> 54  : Recognizer service is starting...
INFO       2024-03-06 17:44:25,717 pika.adapters.utils.connection_workflow start 179 : Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
INFO       2024-03-06 17:44:25,717 pika.adapters.utils.io_services_utils _on_writable 345 : Socket connected: <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 36202), raddr=('127.0.0.1', 5672)>
INFO       2024-03-06 17:44:25,718 pika.adapters.utils.connection_workflow _on_transport_establishment_done 428 : Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9d991ddfc0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9d991ddfc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
INFO       2024-03-06 17:44:25,720 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 293 : AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9d991ddfc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-06 17:44:25,720 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 725 : AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9d991ddfc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-06 17:44:25,720 pika.adapters.blocking_connection _create_connection 453 : Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f9d991ddfc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-06 17:44:25,720 pika.adapters.blocking_connection __init__ 1264: Created channel=1
INFO       2024-03-06 17:44:30,726 __main__ <module> 69  : Connection established.
INFO       2024-03-06 17:45:26,745 __main__ recognize 38  : Recognized person with id 3 and distance 13.062925338745117
INFO       2024-03-06 17:45:26,745 __main__ recognize 38  : Recognized person with id 0 and distance 22.469091415405273
INFO       2024-03-06 17:45:27,071 __main__ recognize 38  : Recognized person with id 3 and distance 14.563117027282715
INFO       2024-03-06 17:45:27,071 __main__ recognize 38  : Recognized person with id 0 and distance 22.726036071777344
INFO       2024-03-06 17:45:27,381 __main__ recognize 38  : Recognized person with id 3 and distance 14.382741928100586
INFO       2024-03-06 17:45:27,381 __main__ recognize 38  : Recognized person with id 0 and distance 22.560577392578125
INFO       2024-03-06 17:45:27,708 __main__ recognize 38  : Recognized person with id 3 and distance 15.195647239685059
INFO       2024-03-06 17:45:27,709 __main__ recognize 38  : Recognized person with id 0 and distance 22.840862274169922
INFO       2024-03-06 17:45:28,030 __main__ recognize 38  : Recognized person with id 3 and distance 15.341602325439453
INFO       2024-03-06 17:45:28,030 __main__ recognize 38  : Recognized person with id 0 and distance 22.48756980895996
INFO       2024-03-06 17:45:28,349 __main__ recognize 38  : Recognized person with id 3 and distance 12.79819107055664
INFO       2024-03-06 17:45:28,349 __main__ recognize 38  : Recognized person with id 0 and distance 22.67211151123047
INFO       2024-03-06 17:45:28,665 __main__ recognize 38  : Recognized person with id 3 and distance 13.544451713562012
INFO       2024-03-06 17:45:28,665 __main__ recognize 38  : Recognized person with id 0 and distance 22.71775245666504
INFO       2024-03-06 17:45:28,988 __main__ recognize 38  : Recognized person with id 3 and distance 13.571455955505371
INFO       2024-03-06 17:45:28,989 __main__ recognize 38  : Recognized person with id 0 and distance 22.45020866394043
INFO       2024-03-06 17:45:29,313 __main__ recognize 38  : Recognized person with id 3 and distance 12.479394912719727
INFO       2024-03-06 17:45:29,313 __main__ recognize 38  : Recognized person with id 0 and distance 22.490385055541992
INFO       2024-03-06 17:45:29,633 __main__ recognize 38  : Recognized person with id 3 and distance 13.12631893157959
INFO       2024-03-06 17:45:29,633 __main__ recognize 38  : Recognized person with id 0 and distance 22.60805892944336
INFO       2024-03-06 17:45:29,950 __main__ recognize 38  : Recognized person with id 3 and distance 11.724462509155273
INFO       2024-03-06 17:45:29,951 __main__ recognize 38  : Recognized person with id 0 and distance 22.496410369873047
INFO       2024-03-06 17:45:30,269 __main__ recognize 38  : Recognized person with id 3 and distance 13.047258377075195
INFO       2024-03-06 17:45:30,269 __main__ recognize 38  : Recognized person with id 0 and distance 22.55161476135254
INFO       2024-03-06 17:45:30,599 __main__ recognize 38  : Recognized person with id 3 and distance 14.27701187133789
INFO       2024-03-06 17:45:30,599 __main__ recognize 38  : Recognized person with id 0 and distance 22.209497451782227
INFO       2024-03-06 17:45:30,931 __main__ recognize 38  : Recognized person with id 3 and distance 11.90886402130127
INFO       2024-03-06 17:45:30,931 __main__ recognize 38  : Recognized person with id 0 and distance 22.54517936706543
INFO       2024-03-06 17:45:31,262 __main__ recognize 38  : Recognized person with id 3 and distance 11.763833999633789
INFO       2024-03-06 17:45:31,262 __main__ recognize 38  : Recognized person with id 0 and distance 22.502361297607422
INFO       2024-03-06 17:45:31,594 __main__ recognize 38  : Recognized person with id 3 and distance 11.421882629394531
INFO       2024-03-06 17:45:31,594 __main__ recognize 38  : Recognized person with id 0 and distance 22.635412216186523
INFO       2024-03-06 17:45:31,925 __main__ recognize 38  : Recognized person with id 3 and distance 12.244218826293945
INFO       2024-03-06 17:45:31,925 __main__ recognize 38  : Recognized person with id 0 and distance 22.674884796142578
INFO       2024-03-06 17:45:32,245 __main__ recognize 38  : Recognized person with id 3 and distance 13.0949068069458
INFO       2024-03-06 17:45:32,245 __main__ recognize 38  : Recognized person with id 0 and distance 22.207592010498047
INFO       2024-03-06 17:45:32,480 __main__ recognize 38  : Recognized person with id 3 and distance 13.518393516540527
INFO       2024-03-06 17:45:32,709 __main__ recognize 38  : Recognized person with id 3 and distance 11.947416305541992
INFO       2024-03-06 17:45:32,941 __main__ recognize 38  : Recognized person with id 3 and distance 11.95893669128418
INFO       2024-03-06 17:45:33,167 __main__ recognize 38  : Recognized person with id 3 and distance 11.867229461669922
INFO       2024-03-06 17:45:33,395 __main__ recognize 38  : Recognized person with id 3 and distance 13.329861640930176
INFO       2024-03-06 17:45:33,630 __main__ recognize 38  : Recognized person with id 3 and distance 13.585200309753418
INFO       2024-03-06 17:45:33,863 __main__ recognize 38  : Recognized person with id 3 and distance 10.91948413848877
INFO       2024-03-06 17:45:34,097 __main__ recognize 38  : Recognized person with id 3 and distance 14.514633178710938
INFO       2024-03-06 17:45:34,357 __main__ recognize 38  : Recognized person with id 3 and distance 11.626591682434082
INFO       2024-03-06 17:45:34,581 __main__ recognize 38  : Recognized person with id 3 and distance 12.100333213806152
INFO       2024-03-06 17:45:34,816 __main__ recognize 38  : Recognized person with id 3 and distance 12.229114532470703
INFO       2024-03-06 17:45:35,046 __main__ recognize 38  : Recognized person with id 3 and distance 12.414510726928711
INFO       2024-03-06 17:45:35,283 __main__ recognize 38  : Recognized person with id 3 and distance 12.7067289352417
INFO       2024-03-06 17:45:35,516 __main__ recognize 38  : Recognized person with id 3 and distance 12.042006492614746
INFO       2024-03-06 17:45:35,753 __main__ recognize 38  : Recognized person with id 3 and distance 14.308874130249023
INFO       2024-03-06 17:45:35,987 __main__ recognize 38  : Recognized person with id 3 and distance 11.71865463256836
INFO       2024-03-06 17:45:36,218 __main__ recognize 38  : Recognized person with id 3 and distance 11.222146987915039
INFO       2024-03-06 17:45:36,450 __main__ recognize 38  : Recognized person with id 3 and distance 12.42358112335205
INFO       2024-03-06 17:45:36,680 __main__ recognize 38  : Recognized person with id 3 and distance 12.294649124145508
INFO       2024-03-06 17:45:36,922 __main__ recognize 38  : Recognized person with id 3 and distance 14.706844329833984
INFO       2024-03-06 17:45:37,149 __main__ recognize 38  : Recognized person with id 3 and distance 11.56399154663086
INFO       2024-03-06 17:45:37,385 __main__ recognize 38  : Recognized person with id 3 and distance 12.914714813232422
INFO       2024-03-06 17:45:37,617 __main__ recognize 38  : Recognized person with id 3 and distance 12.124909400939941
INFO       2024-03-06 17:45:37,854 __main__ recognize 38  : Recognized person with id 3 and distance 12.141905784606934
INFO       2024-03-06 17:45:38,186 __main__ recognize 38  : Recognized person with id 3 and distance 13.011643409729004
INFO       2024-03-06 17:45:38,186 __main__ recognize 38  : Recognized person with id 0 and distance 23.025373458862305
INFO       2024-03-06 17:45:38,418 __main__ recognize 38  : Recognized person with id 3 and distance 12.709271430969238
INFO       2024-03-06 17:45:38,644 __main__ recognize 38  : Recognized person with id 3 and distance 14.415556907653809
INFO       2024-03-06 17:45:38,973 __main__ recognize 38  : Recognized person with id 3 and distance 14.151545524597168
INFO       2024-03-06 17:45:38,973 __main__ recognize 38  : Recognized person with id 0 and distance 23.13355255126953
INFO       2024-03-06 17:45:39,288 __main__ recognize 38  : Recognized person with id 3 and distance 15.4245023727417
INFO       2024-03-06 17:45:39,288 __main__ recognize 38  : Recognized person with id 0 and distance 22.755971908569336
INFO       2024-03-06 17:45:39,606 __main__ recognize 38  : Recognized person with id 3 and distance 12.49570369720459
INFO       2024-03-06 17:45:39,606 __main__ recognize 38  : Recognized person with id 0 and distance 22.762550354003906
INFO       2024-03-06 17:45:39,928 __main__ recognize 38  : Recognized person with id 3 and distance 11.597892761230469
INFO       2024-03-06 17:45:39,928 __main__ recognize 38  : Recognized person with id 0 and distance 22.695829391479492
INFO       2024-03-06 17:45:40,248 __main__ recognize 38  : Recognized person with id 3 and distance 11.819336891174316
INFO       2024-03-06 17:45:40,248 __main__ recognize 38  : Recognized person with id 0 and distance 22.74999237060547
INFO       2024-03-06 17:46:51,527 __main__ <module> 54  : Recognizer service is starting...
INFO       2024-03-06 17:46:51,529 pika.adapters.utils.connection_workflow start 179 : Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
INFO       2024-03-06 17:46:51,529 pika.adapters.utils.io_services_utils _on_writable 345 : Socket connected: <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 38312), raddr=('127.0.0.1', 5672)>
INFO       2024-03-06 17:46:51,529 pika.adapters.utils.connection_workflow _on_transport_establishment_done 428 : Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f155d3d1fc0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f155d3d1fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
INFO       2024-03-06 17:46:51,531 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 293 : AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f155d3d1fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-06 17:46:51,532 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 725 : AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f155d3d1fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-06 17:46:51,532 pika.adapters.blocking_connection _create_connection 453 : Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f155d3d1fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-06 17:46:51,532 pika.adapters.blocking_connection __init__ 1264: Created channel=1
INFO       2024-03-06 17:46:56,541 __main__ <module> 69  : Connection established.
INFO       2024-03-06 17:47:14,457 __main__ recognize 38  : Recognized person with id 3 and distance 14.236248016357422
INFO       2024-03-06 17:47:14,457 __main__ recognize 38  : Recognized person with id 0 and distance 22.831012725830078
INFO       2024-03-06 17:47:14,804 __main__ recognize 38  : Recognized person with id 3 and distance 15.866694450378418
INFO       2024-03-06 17:47:14,804 __main__ recognize 38  : Recognized person with id 0 and distance 22.95846939086914
INFO       2024-03-06 17:47:15,125 __main__ recognize 38  : Recognized person with id 3 and distance 17.264217376708984
INFO       2024-03-06 17:47:15,125 __main__ recognize 38  : Recognized person with id 0 and distance 22.606060028076172
INFO       2024-03-06 17:47:15,456 __main__ recognize 38  : Recognized person with id 3 and distance 19.432958602905273
INFO       2024-03-06 17:47:15,457 __main__ recognize 38  : Recognized person with id 0 and distance 23.075950622558594
INFO       2024-03-06 17:47:15,691 __main__ recognize 38  : Recognized person with id 3 and distance 19.685401916503906
INFO       2024-03-06 17:47:16,014 __main__ recognize 38  : Recognized person with id 3 and distance 16.283050537109375
INFO       2024-03-06 17:47:16,014 __main__ recognize 38  : Recognized person with id 0 and distance 22.853618621826172
INFO       2024-03-06 17:47:16,357 __main__ recognize 38  : Recognized person with id 3 and distance 15.720499992370605
INFO       2024-03-06 17:47:16,357 __main__ recognize 38  : Recognized person with id 0 and distance 22.723237991333008
INFO       2024-03-06 17:47:16,703 __main__ recognize 38  : Recognized person with id 3 and distance 14.595428466796875
INFO       2024-03-06 17:47:16,704 __main__ recognize 38  : Recognized person with id 0 and distance 22.72361946105957
INFO       2024-03-06 17:47:17,031 __main__ recognize 38  : Recognized person with id 3 and distance 13.734626770019531
INFO       2024-03-06 17:47:17,031 __main__ recognize 38  : Recognized person with id 0 and distance 22.773717880249023
INFO       2024-03-06 17:47:17,359 __main__ recognize 38  : Recognized person with id 3 and distance 14.108810424804688
INFO       2024-03-06 17:47:17,359 __main__ recognize 38  : Recognized person with id 0 and distance 23.092069625854492
INFO       2024-03-06 17:47:17,686 __main__ recognize 38  : Recognized person with id 3 and distance 15.127294540405273
INFO       2024-03-06 17:47:17,686 __main__ recognize 38  : Recognized person with id 0 and distance 22.764625549316406
INFO       2024-03-06 17:47:18,016 __main__ recognize 38  : Recognized person with id 3 and distance 13.556548118591309
INFO       2024-03-06 17:47:18,017 __main__ recognize 38  : Recognized person with id 0 and distance 22.97031593322754
INFO       2024-03-06 17:47:18,346 __main__ recognize 38  : Recognized person with id 3 and distance 13.53695011138916
INFO       2024-03-06 17:47:18,346 __main__ recognize 38  : Recognized person with id 0 and distance 22.77042579650879
INFO       2024-03-06 17:47:18,671 __main__ recognize 38  : Recognized person with id 3 and distance 14.51419448852539
INFO       2024-03-06 17:47:18,671 __main__ recognize 38  : Recognized person with id 0 and distance 22.9932861328125
INFO       2024-03-06 17:47:18,999 __main__ recognize 38  : Recognized person with id 3 and distance 13.376360893249512
INFO       2024-03-06 17:47:18,999 __main__ recognize 38  : Recognized person with id 0 and distance 22.96080207824707
INFO       2024-03-06 17:47:19,323 __main__ recognize 38  : Recognized person with id 3 and distance 12.76539134979248
INFO       2024-03-06 17:47:19,323 __main__ recognize 38  : Recognized person with id 0 and distance 22.809329986572266
INFO       2024-03-06 17:47:19,649 __main__ recognize 38  : Recognized person with id 3 and distance 12.668445587158203
INFO       2024-03-06 17:47:19,649 __main__ recognize 38  : Recognized person with id 0 and distance 23.07500457763672
INFO       2024-03-06 17:47:19,977 __main__ recognize 38  : Recognized person with id 3 and distance 14.477017402648926
INFO       2024-03-06 17:47:19,977 __main__ recognize 38  : Recognized person with id 3 and distance 23.039424896240234
INFO       2024-03-06 17:47:20,305 __main__ recognize 38  : Recognized person with id 3 and distance 12.491908073425293
INFO       2024-03-06 17:47:20,305 __main__ recognize 38  : Recognized person with id 0 and distance 22.772918701171875
INFO       2024-03-06 17:47:20,621 __main__ recognize 38  : Recognized person with id 3 and distance 11.53251838684082
INFO       2024-03-06 17:47:20,621 __main__ recognize 38  : Recognized person with id 0 and distance 23.02530288696289
INFO       2024-03-06 17:47:20,944 __main__ recognize 38  : Recognized person with id 3 and distance 11.880071640014648
INFO       2024-03-06 17:47:20,944 __main__ recognize 38  : Recognized person with id 0 and distance 23.079246520996094
INFO       2024-03-06 17:47:21,269 __main__ recognize 38  : Recognized person with id 3 and distance 13.029452323913574
INFO       2024-03-06 17:47:21,269 __main__ recognize 38  : Recognized person with id 0 and distance 23.104352951049805
INFO       2024-03-06 17:47:21,597 __main__ recognize 38  : Recognized person with id 3 and distance 13.712652206420898
INFO       2024-03-06 17:47:21,597 __main__ recognize 38  : Recognized person with id 0 and distance 22.959291458129883
INFO       2024-03-06 17:47:21,923 __main__ recognize 38  : Recognized person with id 3 and distance 11.990528106689453
INFO       2024-03-06 17:47:21,923 __main__ recognize 38  : Recognized person with id 0 and distance 22.934616088867188
INFO       2024-03-06 17:47:22,251 __main__ recognize 38  : Recognized person with id 3 and distance 13.767924308776855
INFO       2024-03-06 17:47:22,251 __main__ recognize 38  : Recognized person with id 0 and distance 22.81495475769043
INFO       2024-03-06 17:47:22,579 __main__ recognize 38  : Recognized person with id 3 and distance 12.878484725952148
INFO       2024-03-06 17:47:22,580 __main__ recognize 38  : Recognized person with id 0 and distance 23.116615295410156
INFO       2024-03-06 17:47:22,906 __main__ recognize 38  : Recognized person with id 3 and distance 12.965463638305664
INFO       2024-03-06 17:47:22,906 __main__ recognize 38  : Recognized person with id 0 and distance 22.911251068115234
INFO       2024-03-06 17:47:23,233 __main__ recognize 38  : Recognized person with id 3 and distance 12.52765941619873
INFO       2024-03-06 17:47:23,234 __main__ recognize 38  : Recognized person with id 0 and distance 22.971641540527344
INFO       2024-03-06 17:47:23,562 __main__ recognize 38  : Recognized person with id 3 and distance 14.615154266357422
INFO       2024-03-06 17:47:23,562 __main__ recognize 38  : Recognized person with id 0 and distance 23.078514099121094
INFO       2024-03-06 17:47:23,890 __main__ recognize 38  : Recognized person with id 3 and distance 12.982926368713379
INFO       2024-03-06 17:47:23,891 __main__ recognize 38  : Recognized person with id 0 and distance 23.065183639526367
INFO       2024-03-06 17:47:24,214 __main__ recognize 38  : Recognized person with id 3 and distance 13.231531143188477
INFO       2024-03-06 17:47:24,214 __main__ recognize 38  : Recognized person with id 0 and distance 22.903743743896484
INFO       2024-03-06 17:47:24,542 __main__ recognize 38  : Recognized person with id 3 and distance 13.410774230957031
INFO       2024-03-06 17:47:24,542 __main__ recognize 38  : Recognized person with id 0 and distance 22.974891662597656
INFO       2024-03-06 17:47:24,868 __main__ recognize 38  : Recognized person with id 3 and distance 14.68101692199707
INFO       2024-03-06 17:47:24,868 __main__ recognize 38  : Recognized person with id 0 and distance 23.16267967224121
INFO       2024-03-06 17:47:25,208 __main__ recognize 38  : Recognized person with id 3 and distance 13.367264747619629
INFO       2024-03-06 17:47:25,208 __main__ recognize 38  : Recognized person with id 0 and distance 23.122827529907227
INFO       2024-03-06 17:47:25,538 __main__ recognize 38  : Recognized person with id 3 and distance 13.100199699401855
INFO       2024-03-06 17:47:25,538 __main__ recognize 38  : Recognized person with id 0 and distance 23.076173782348633
INFO       2024-03-06 17:47:25,869 __main__ recognize 38  : Recognized person with id 3 and distance 14.274201393127441
INFO       2024-03-06 17:47:25,869 __main__ recognize 38  : Recognized person with id 0 and distance 23.062255859375
INFO       2024-03-06 17:47:26,196 __main__ recognize 38  : Recognized person with id 3 and distance 14.654419898986816
INFO       2024-03-06 17:47:26,196 __main__ recognize 38  : Recognized person with id 0 and distance 22.85409927368164
INFO       2024-03-06 17:47:26,528 __main__ recognize 38  : Recognized person with id 3 and distance 15.415905952453613
INFO       2024-03-06 17:47:26,528 __main__ recognize 38  : Recognized person with id 0 and distance 22.86440658569336
INFO       2024-03-06 17:47:26,859 __main__ recognize 38  : Recognized person with id 3 and distance 13.43237590789795
INFO       2024-03-06 17:47:26,859 __main__ recognize 38  : Recognized person with id 0 and distance 23.10930061340332
INFO       2024-03-06 17:47:27,186 __main__ recognize 38  : Recognized person with id 3 and distance 13.782880783081055
INFO       2024-03-06 17:47:27,186 __main__ recognize 38  : Recognized person with id 0 and distance 23.33670997619629
INFO       2024-03-06 17:47:27,511 __main__ recognize 38  : Recognized person with id 3 and distance 14.607904434204102
INFO       2024-03-06 17:47:27,511 __main__ recognize 38  : Recognized person with id 0 and distance 23.12105369567871
INFO       2024-03-06 17:47:27,842 __main__ recognize 38  : Recognized person with id 3 and distance 14.433011054992676
INFO       2024-03-06 17:47:27,843 __main__ recognize 38  : Recognized person with id 0 and distance 23.156566619873047
INFO       2024-03-06 17:47:28,171 __main__ recognize 38  : Recognized person with id 3 and distance 14.274616241455078
INFO       2024-03-06 17:47:28,172 __main__ recognize 38  : Recognized person with id 0 and distance 23.137296676635742
INFO       2024-03-06 17:47:28,499 __main__ recognize 38  : Recognized person with id 3 and distance 13.815258026123047
INFO       2024-03-06 17:47:28,500 __main__ recognize 38  : Recognized person with id 0 and distance 23.226160049438477
INFO       2024-03-06 17:50:49,468 __main__ recognize 38  : Recognized person with id 3 and distance 14.269073486328125
INFO       2024-03-06 17:50:49,468 __main__ recognize 38  : Recognized person with id 0 and distance 23.026535034179688
INFO       2024-03-06 17:50:49,796 __main__ recognize 38  : Recognized person with id 3 and distance 12.34787368774414
INFO       2024-03-06 17:50:49,796 __main__ recognize 38  : Recognized person with id 0 and distance 22.8637638092041
INFO       2024-03-06 17:50:50,125 __main__ recognize 38  : Recognized person with id 3 and distance 12.985796928405762
INFO       2024-03-06 17:50:50,125 __main__ recognize 38  : Recognized person with id 0 and distance 23.033079147338867
INFO       2024-03-06 17:50:50,443 __main__ recognize 38  : Recognized person with id 3 and distance 12.89037036895752
INFO       2024-03-06 17:50:50,443 __main__ recognize 38  : Recognized person with id 0 and distance 22.802419662475586
INFO       2024-03-06 17:50:50,754 __main__ recognize 38  : Recognized person with id 3 and distance 12.962932586669922
INFO       2024-03-06 17:50:50,754 __main__ recognize 38  : Recognized person with id 0 and distance 22.832582473754883
INFO       2024-03-06 17:50:50,984 __main__ recognize 38  : Recognized person with id 3 and distance 12.516524314880371
INFO       2024-03-06 17:50:51,212 __main__ recognize 38  : Recognized person with id 3 and distance 13.848368644714355
INFO       2024-03-06 17:50:51,440 __main__ recognize 38  : Recognized person with id 3 and distance 10.681062698364258
INFO       2024-03-06 17:50:51,674 __main__ recognize 38  : Recognized person with id 3 and distance 10.69250774383545
INFO       2024-03-06 17:50:51,901 __main__ recognize 38  : Recognized person with id 3 and distance 14.314697265625
INFO       2024-03-06 17:50:52,132 __main__ recognize 38  : Recognized person with id 3 and distance 13.932027816772461
INFO       2024-03-06 17:50:52,357 __main__ recognize 38  : Recognized person with id 3 and distance 12.524676322937012
INFO       2024-03-06 17:50:52,587 __main__ recognize 38  : Recognized person with id 3 and distance 13.260419845581055
INFO       2024-03-06 17:50:52,813 __main__ recognize 38  : Recognized person with id 3 and distance 13.288411140441895
INFO       2024-03-06 17:50:53,044 __main__ recognize 38  : Recognized person with id 3 and distance 13.163869857788086
INFO       2024-03-06 17:50:53,274 __main__ recognize 38  : Recognized person with id 3 and distance 15.012452125549316
INFO       2024-03-06 17:50:53,504 __main__ recognize 38  : Recognized person with id 3 and distance 11.633820533752441
INFO       2024-03-06 17:50:53,732 __main__ recognize 38  : Recognized person with id 3 and distance 11.204337120056152
INFO       2024-03-06 17:50:53,961 __main__ recognize 38  : Recognized person with id 3 and distance 13.939496994018555
INFO       2024-03-06 17:50:54,188 __main__ recognize 38  : Recognized person with id 3 and distance 13.543086051940918
INFO       2024-03-06 17:50:54,410 __main__ recognize 38  : Recognized person with id 3 and distance 14.321914672851562
INFO       2024-03-06 17:50:54,641 __main__ recognize 38  : Recognized person with id 3 and distance 12.819397926330566
INFO       2024-03-06 17:50:54,879 __main__ recognize 38  : Recognized person with id 3 and distance 13.29350757598877
INFO       2024-03-06 17:50:55,108 __main__ recognize 38  : Recognized person with id 3 and distance 11.478447914123535
INFO       2024-03-06 17:50:55,340 __main__ recognize 38  : Recognized person with id 3 and distance 14.706039428710938
INFO       2024-03-06 17:50:55,574 __main__ recognize 38  : Recognized person with id 3 and distance 10.965126991271973
INFO       2024-03-06 17:50:55,808 __main__ recognize 38  : Recognized person with id 3 and distance 10.698246002197266
INFO       2024-03-06 17:50:56,036 __main__ recognize 38  : Recognized person with id 3 and distance 13.687726020812988
INFO       2024-03-06 17:50:56,268 __main__ recognize 38  : Recognized person with id 3 and distance 13.096551895141602
INFO       2024-03-06 17:50:56,503 __main__ recognize 38  : Recognized person with id 3 and distance 12.283529281616211
INFO       2024-03-06 17:50:56,737 __main__ recognize 38  : Recognized person with id 3 and distance 11.694574356079102
INFO       2024-03-06 17:50:56,971 __main__ recognize 38  : Recognized person with id 3 and distance 11.356025695800781
INFO       2024-03-06 17:50:57,208 __main__ recognize 38  : Recognized person with id 3 and distance 10.168438911437988
INFO       2024-03-06 17:50:57,440 __main__ recognize 38  : Recognized person with id 3 and distance 13.02528190612793
INFO       2024-03-06 17:50:57,672 __main__ recognize 38  : Recognized person with id 3 and distance 11.239858627319336
INFO       2024-03-06 17:50:57,901 __main__ recognize 38  : Recognized person with id 3 and distance 11.904918670654297
INFO       2024-03-06 17:50:58,133 __main__ recognize 38  : Recognized person with id 3 and distance 12.046721458435059
INFO       2024-03-06 17:50:58,364 __main__ recognize 38  : Recognized person with id 3 and distance 13.284314155578613
INFO       2024-03-06 17:50:58,596 __main__ recognize 38  : Recognized person with id 3 and distance 12.312328338623047
INFO       2024-03-06 17:50:58,833 __main__ recognize 38  : Recognized person with id 3 and distance 12.22638988494873
INFO       2024-03-06 17:50:59,069 __main__ recognize 38  : Recognized person with id 3 and distance 12.745348930358887
INFO       2024-03-06 17:50:59,293 __main__ recognize 38  : Recognized person with id 3 and distance 13.796666145324707
INFO       2024-03-06 17:50:59,522 __main__ recognize 38  : Recognized person with id 3 and distance 10.90130615234375
INFO       2024-03-06 17:50:59,752 __main__ recognize 38  : Recognized person with id 3 and distance 12.495744705200195
INFO       2024-03-06 17:50:59,988 __main__ recognize 38  : Recognized person with id 3 and distance 12.50635051727295
INFO       2024-03-06 17:51:00,221 __main__ recognize 38  : Recognized person with id 3 and distance 18.738500595092773
INFO       2024-03-06 17:51:06,000 pika.adapters.blocking_connection close 802 : Closing connection (200): Normal shutdown
INFO       2024-03-06 17:51:06,002 pika.channel close 536 : Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f155d3d1fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
INFO       2024-03-06 17:51:06,003 pika.channel _on_closeok 1133: Received <Channel.CloseOk> on <Channel number=1 CLOSING conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f155d3d1fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
INFO       2024-03-06 17:51:06,003 pika.connection close 1316: Closing connection (200): 'Normal shutdown'
INFO       2024-03-06 17:51:06,004 pika.adapters.utils.io_services_utils abort 731 : Aborting transport connection: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 38312), raddr=('127.0.0.1', 5672)>
INFO       2024-03-06 17:51:06,005 pika.adapters.utils.io_services_utils _initiate_abort 904 : _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 38312), raddr=('127.0.0.1', 5672)>
INFO       2024-03-06 17:51:06,005 pika.adapters.utils.io_services_utils _deactivate 869 : Deactivating transport: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 38312), raddr=('127.0.0.1', 5672)>
INFO       2024-03-06 17:51:06,005 pika.connection _on_stream_terminated 2017: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
INFO       2024-03-06 17:51:06,005 pika.connection _on_stream_terminated 2086: Stack terminated due to ConnectionClosedByClient: (200) 'Normal shutdown'
INFO       2024-03-06 17:51:06,005 pika.adapters.utils.io_services_utils _close_and_finalize 882 : Closing transport socket and unlinking: state=3; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 38312), raddr=('127.0.0.1', 5672)>
INFO       2024-03-06 17:51:06,005 pika.adapters.blocking_connection _flush_output 525 : User-initiated close: result=BlockingConnection__OnClosedArgs(connection=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>, error=ConnectionClosedByClient: (200) 'Normal shutdown')
INFO       2024-03-06 17:51:06,005 __main__ <module> 76  : Connection closed. Application closing.
INFO       2024-03-06 17:51:07,068 __main__ <module> 54  : Recognizer service is starting...
INFO       2024-03-06 17:51:07,070 pika.adapters.utils.connection_workflow start 179 : Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
INFO       2024-03-06 17:51:07,070 pika.adapters.utils.io_services_utils _on_writable 345 : Socket connected: <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 34538), raddr=('127.0.0.1', 5672)>
INFO       2024-03-06 17:51:07,070 pika.adapters.utils.connection_workflow _on_transport_establishment_done 428 : Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7ff91ddfc0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7ff91ddfc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
INFO       2024-03-06 17:51:07,072 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 293 : AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7ff91ddfc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-06 17:51:07,072 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 725 : AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7ff91ddfc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-06 17:51:07,072 pika.adapters.blocking_connection _create_connection 453 : Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f7ff91ddfc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-06 17:51:07,072 pika.adapters.blocking_connection __init__ 1264: Created channel=1
INFO       2024-03-06 17:51:12,078 __main__ <module> 69  : Connection established.
INFO       2024-03-06 17:51:27,626 __main__ recognize 38  : Recognized person with id 3 and distance 14.757346153259277
INFO       2024-03-06 17:51:27,870 __main__ recognize 38  : Recognized person with id 3 and distance 13.471308708190918
INFO       2024-03-06 17:51:28,104 __main__ recognize 38  : Recognized person with id 3 and distance 13.777280807495117
INFO       2024-03-06 17:51:28,340 __main__ recognize 38  : Recognized person with id 3 and distance 13.909937858581543
INFO       2024-03-06 17:51:28,576 __main__ recognize 38  : Recognized person with id 3 and distance 12.687264442443848
INFO       2024-03-06 17:51:28,941 __main__ recognize 38  : Recognized person with id 3 and distance 18.21971893310547
INFO       2024-03-06 17:51:29,174 __main__ recognize 38  : Recognized person with id 3 and distance 15.86215877532959
INFO       2024-03-06 17:51:29,405 __main__ recognize 38  : Recognized person with id 3 and distance 18.680191040039062
INFO       2024-03-06 17:51:29,637 __main__ recognize 38  : Recognized person with id 3 and distance 19.993173599243164
INFO       2024-03-06 17:51:29,874 __main__ recognize 38  : Recognized person with id 3 and distance 14.588105201721191
INFO       2024-03-06 17:51:30,098 __main__ recognize 38  : Recognized person with id 3 and distance 15.86290168762207
INFO       2024-03-06 17:51:30,324 __main__ recognize 38  : Recognized person with id 3 and distance 24.338481903076172
INFO       2024-03-06 17:51:30,557 __main__ recognize 38  : Recognized person with id 3 and distance 16.385011672973633
INFO       2024-03-06 17:51:30,880 __main__ recognize 38  : Recognized person with id 3 and distance 13.454066276550293
INFO       2024-03-06 17:51:30,880 __main__ recognize 38  : Recognized person with id 0 and distance 22.732831954956055
INFO       2024-03-06 17:51:31,107 __main__ recognize 38  : Recognized person with id 3 and distance 15.870911598205566
INFO       2024-03-06 17:51:31,334 __main__ recognize 38  : Recognized person with id 3 and distance 13.14813232421875
INFO       2024-03-06 17:51:31,564 __main__ recognize 38  : Recognized person with id 3 and distance 13.422930717468262
INFO       2024-03-06 17:51:31,794 __main__ recognize 38  : Recognized person with id 3 and distance 12.320396423339844
INFO       2024-03-06 17:51:32,019 __main__ recognize 38  : Recognized person with id 3 and distance 12.54476261138916
INFO       2024-03-06 17:51:32,248 __main__ recognize 38  : Recognized person with id 3 and distance 12.766866683959961
INFO       2024-03-06 17:51:32,477 __main__ recognize 38  : Recognized person with id 3 and distance 12.6812162399292
INFO       2024-03-06 17:51:32,706 __main__ recognize 38  : Recognized person with id 3 and distance 11.629838943481445
INFO       2024-03-06 17:51:32,940 __main__ recognize 38  : Recognized person with id 3 and distance 12.117571830749512
INFO       2024-03-06 17:51:33,176 __main__ recognize 38  : Recognized person with id 3 and distance 12.213218688964844
INFO       2024-03-06 17:51:33,408 __main__ recognize 38  : Recognized person with id 3 and distance 12.98641586303711
INFO       2024-03-06 17:51:33,644 __main__ recognize 38  : Recognized person with id 3 and distance 12.630523681640625
INFO       2024-03-06 17:51:33,873 __main__ recognize 38  : Recognized person with id 3 and distance 13.342989921569824
INFO       2024-03-06 17:51:34,098 __main__ recognize 38  : Recognized person with id 3 and distance 14.053812026977539
INFO       2024-03-06 17:51:34,329 __main__ recognize 38  : Recognized person with id 3 and distance 15.053815841674805
INFO       2024-03-06 17:51:34,557 __main__ recognize 38  : Recognized person with id 3 and distance 12.519576072692871
INFO       2024-03-06 17:51:34,788 __main__ recognize 38  : Recognized person with id 3 and distance 12.301941871643066
INFO       2024-03-06 17:51:35,019 __main__ recognize 38  : Recognized person with id 3 and distance 12.914763450622559
INFO       2024-03-06 17:51:35,247 __main__ recognize 38  : Recognized person with id 3 and distance 12.433625221252441
INFO       2024-03-06 17:51:35,476 __main__ recognize 38  : Recognized person with id 3 and distance 13.110126495361328
INFO       2024-03-06 17:51:35,703 __main__ recognize 38  : Recognized person with id 3 and distance 14.07827377319336
INFO       2024-03-06 17:51:35,935 __main__ recognize 38  : Recognized person with id 3 and distance 12.31295108795166
INFO       2024-03-06 17:51:36,170 __main__ recognize 38  : Recognized person with id 3 and distance 13.276786804199219
INFO       2024-03-06 17:51:36,408 __main__ recognize 38  : Recognized person with id 3 and distance 15.167898178100586
INFO       2024-03-06 17:51:36,643 __main__ recognize 38  : Recognized person with id 3 and distance 13.484492301940918
INFO       2024-03-06 17:51:36,876 __main__ recognize 38  : Recognized person with id 3 and distance 14.684296607971191
INFO       2024-03-06 17:51:37,125 __main__ recognize 38  : Recognized person with id 3 and distance 14.478899002075195
INFO       2024-03-06 19:39:21,088 pika.adapters.utils.io_services_utils abort 731 : Aborting transport connection: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 34538), raddr=('127.0.0.1', 5672)>
INFO       2024-03-06 19:39:21,088 pika.adapters.utils.io_services_utils _initiate_abort 904 : _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 34538), raddr=('127.0.0.1', 5672)>
INFO       2024-03-06 19:39:21,088 pika.adapters.utils.io_services_utils _deactivate 869 : Deactivating transport: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 34538), raddr=('127.0.0.1', 5672)>
INFO       2024-03-06 19:39:21,088 pika.connection _on_stream_terminated 2017: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
INFO       2024-03-06 19:39:21,089 pika.connection _on_stream_terminated 2086: Stack terminated due to ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
INFO       2024-03-06 19:39:21,089 pika.adapters.utils.io_services_utils _close_and_finalize 882 : Closing transport socket and unlinking: state=3; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 34538), raddr=('127.0.0.1', 5672)>
ERROR      2024-03-06 19:39:21,089 pika.adapters.blocking_connection _flush_output 521 : Unexpected connection close detected: ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
INFO       2024-03-07 18:45:25,972 __main__ <module> 54  : Recognizer service is starting...
INFO       2024-03-07 18:45:25,980 pika.adapters.utils.connection_workflow start 179 : Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
INFO       2024-03-07 18:45:25,982 pika.adapters.utils.io_services_utils _on_writable 345 : Socket connected: <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 44268), raddr=('127.0.0.1', 5672)>
INFO       2024-03-07 18:45:25,984 pika.adapters.utils.connection_workflow _on_transport_establishment_done 428 : Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fb7ce6476a0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fb7ce6476a0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
INFO       2024-03-07 18:45:25,993 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 293 : AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fb7ce6476a0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-07 18:45:25,994 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 725 : AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fb7ce6476a0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-07 18:45:25,994 pika.adapters.blocking_connection _create_connection 453 : Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fb7ce6476a0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-07 18:45:25,995 pika.adapters.blocking_connection __init__ 1264: Created channel=1
INFO       2024-03-07 18:45:31,038 __main__ <module> 69  : Connection established.
INFO       2024-03-07 18:46:41,955 pika.adapters.blocking_connection close 802 : Closing connection (200): Normal shutdown
INFO       2024-03-07 18:46:41,957 pika.channel close 536 : Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fb7ce6476a0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
INFO       2024-03-07 18:46:41,958 pika.channel _on_closeok 1133: Received <Channel.CloseOk> on <Channel number=1 CLOSING conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7fb7ce6476a0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
INFO       2024-03-07 18:46:41,959 pika.connection close 1316: Closing connection (200): 'Normal shutdown'
INFO       2024-03-07 18:46:41,960 pika.adapters.utils.io_services_utils abort 731 : Aborting transport connection: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 44268), raddr=('127.0.0.1', 5672)>
INFO       2024-03-07 18:46:41,960 pika.adapters.utils.io_services_utils _initiate_abort 904 : _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 44268), raddr=('127.0.0.1', 5672)>
INFO       2024-03-07 18:46:41,960 pika.adapters.utils.io_services_utils _deactivate 869 : Deactivating transport: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 44268), raddr=('127.0.0.1', 5672)>
INFO       2024-03-07 18:46:41,961 pika.connection _on_stream_terminated 2017: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
INFO       2024-03-07 18:46:41,961 pika.connection _on_stream_terminated 2086: Stack terminated due to ConnectionClosedByClient: (200) 'Normal shutdown'
INFO       2024-03-07 18:46:41,961 pika.adapters.utils.io_services_utils _close_and_finalize 882 : Closing transport socket and unlinking: state=3; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 44268), raddr=('127.0.0.1', 5672)>
INFO       2024-03-07 18:46:41,961 pika.adapters.blocking_connection _flush_output 525 : User-initiated close: result=BlockingConnection__OnClosedArgs(connection=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>, error=ConnectionClosedByClient: (200) 'Normal shutdown')
INFO       2024-03-07 18:46:41,962 __main__ <module> 76  : Connection closed. Application closing.
INFO       2024-03-07 18:46:52,613 __main__ <module> 54  : Recognizer service is starting...
INFO       2024-03-07 18:46:52,615 pika.adapters.utils.connection_workflow start 179 : Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
INFO       2024-03-07 18:46:52,615 pika.adapters.utils.io_services_utils _on_writable 345 : Socket connected: <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 38504), raddr=('127.0.0.1', 5672)>
INFO       2024-03-07 18:46:52,615 pika.adapters.utils.connection_workflow _on_transport_establishment_done 428 : Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f19f83a9fc0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f19f83a9fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
INFO       2024-03-07 18:46:52,619 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 293 : AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f19f83a9fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-07 18:46:52,619 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 725 : AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f19f83a9fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-07 18:46:52,619 pika.adapters.blocking_connection _create_connection 453 : Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7f19f83a9fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-07 18:46:52,619 pika.adapters.blocking_connection __init__ 1264: Created channel=1
INFO       2024-03-07 18:46:57,630 __main__ <module> 69  : Connection established.
INFO       2024-03-07 18:48:09,117 __main__ recognize 38  : Recognized person with id 0 and distance 23.649677276611328
INFO       2024-03-07 18:48:09,118 __main__ recognize 38  : Recognized person with id 3 and distance 17.009132385253906
INFO       2024-03-07 18:48:09,478 __main__ recognize 38  : Recognized person with id 3 and distance 16.173198699951172
INFO       2024-03-07 18:48:09,932 __main__ recognize 38  : Recognized person with id 3 and distance 23.42035484313965
INFO       2024-03-07 18:48:09,932 __main__ recognize 38  : Recognized person with id 3 and distance 15.938094139099121
INFO       2024-03-07 18:48:10,374 __main__ recognize 38  : Recognized person with id 3 and distance 15.414314270019531
INFO       2024-03-07 18:48:10,782 __main__ recognize 38  : Recognized person with id 3 and distance 15.674887657165527
INFO       2024-03-07 18:48:10,783 __main__ recognize 38  : Recognized person with id 3 and distance 23.356904983520508
INFO       2024-03-07 18:48:11,308 __main__ recognize 38  : Recognized person with id 3 and distance 16.982444763183594
INFO       2024-03-07 18:48:11,907 __main__ recognize 38  : Recognized person with id 3 and distance 16.713560104370117
INFO       2024-03-07 18:48:11,907 __main__ recognize 38  : Recognized person with id 3 and distance 23.32732391357422
INFO       2024-03-07 18:48:12,268 __main__ recognize 38  : Recognized person with id 3 and distance 18.946834564208984
INFO       2024-03-07 18:48:12,795 __main__ recognize 38  : Recognized person with id 3 and distance 16.260669708251953
INFO       2024-03-07 18:48:12,795 __main__ recognize 38  : Recognized person with id 3 and distance 23.326457977294922
INFO       2024-03-07 18:48:13,279 __main__ recognize 38  : Recognized person with id 3 and distance 15.0004243850708
INFO       2024-03-07 18:48:13,279 __main__ recognize 38  : Recognized person with id 3 and distance 23.347375869750977
INFO       2024-03-07 18:48:13,846 __main__ recognize 38  : Recognized person with id 3 and distance 16.992231369018555
INFO       2024-03-07 18:48:13,846 __main__ recognize 38  : Recognized person with id 3 and distance 23.306074142456055
INFO       2024-03-07 18:48:14,413 __main__ recognize 38  : Recognized person with id 3 and distance 15.055808067321777
INFO       2024-03-07 18:48:14,414 __main__ recognize 38  : Recognized person with id 3 and distance 23.306509017944336
INFO       2024-03-07 18:48:14,972 __main__ recognize 38  : Recognized person with id 3 and distance 14.980551719665527
INFO       2024-03-07 18:48:15,464 __main__ recognize 38  : Recognized person with id 3 and distance 15.086167335510254
INFO       2024-03-07 18:48:15,464 __main__ recognize 38  : Recognized person with id 3 and distance 23.521007537841797
INFO       2024-03-07 18:48:16,068 __main__ recognize 38  : Recognized person with id 3 and distance 15.070852279663086
INFO       2024-03-07 18:48:16,068 __main__ recognize 38  : Recognized person with id 3 and distance 23.497243881225586
INFO       2024-03-07 18:48:16,655 __main__ recognize 38  : Recognized person with id 3 and distance 16.243425369262695
INFO       2024-03-07 18:48:16,655 __main__ recognize 38  : Recognized person with id 3 and distance 23.437339782714844
INFO       2024-03-07 18:48:17,026 __main__ recognize 38  : Recognized person with id 3 and distance 15.60441780090332
INFO       2024-03-07 18:48:17,520 __main__ recognize 38  : Recognized person with id 3 and distance 15.937597274780273
INFO       2024-03-07 18:48:17,520 __main__ recognize 38  : Recognized person with id 3 and distance 23.212032318115234
INFO       2024-03-07 18:48:17,960 __main__ recognize 38  : Recognized person with id 3 and distance 16.097938537597656
INFO       2024-03-07 18:48:17,961 __main__ recognize 38  : Recognized person with id 3 and distance 23.447702407836914
INFO       2024-03-07 19:45:07,072 pika.adapters.utils.io_services_utils abort 731 : Aborting transport connection: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 38504), raddr=('127.0.0.1', 5672)>
INFO       2024-03-07 19:45:07,072 pika.adapters.utils.io_services_utils _initiate_abort 904 : _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 38504), raddr=('127.0.0.1', 5672)>
INFO       2024-03-07 19:45:07,072 pika.adapters.utils.io_services_utils _deactivate 869 : Deactivating transport: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 38504), raddr=('127.0.0.1', 5672)>
INFO       2024-03-07 19:45:07,073 pika.connection _on_stream_terminated 2017: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
INFO       2024-03-07 19:45:07,073 pika.connection _on_stream_terminated 2086: Stack terminated due to ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
INFO       2024-03-07 19:45:07,073 pika.adapters.utils.io_services_utils _close_and_finalize 882 : Closing transport socket and unlinking: state=3; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 38504), raddr=('127.0.0.1', 5672)>
ERROR      2024-03-07 19:45:07,073 pika.adapters.blocking_connection _flush_output 521 : Unexpected connection close detected: ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
INFO       2024-03-13 18:31:51,092 __main__ <module> 54  : Recognizer service is starting...
INFO       2024-03-13 18:31:51,095 pika.adapters.utils.connection_workflow start 179 : Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
INFO       2024-03-13 18:31:51,095 pika.adapters.utils.io_services_utils _on_writable 345 : Socket connected: <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 34224), raddr=('127.0.0.1', 5672)>
INFO       2024-03-13 18:31:51,096 pika.adapters.utils.connection_workflow _on_transport_establishment_done 428 : Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7372be3d1fc0>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7372be3d1fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
INFO       2024-03-13 18:31:51,100 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 293 : AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7372be3d1fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-13 18:31:51,100 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 725 : AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7372be3d1fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-13 18:31:51,100 pika.adapters.blocking_connection _create_connection 453 : Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7372be3d1fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-13 18:31:51,101 pika.adapters.blocking_connection __init__ 1264: Created channel=1
INFO       2024-03-13 18:31:56,123 __main__ <module> 69  : Connection established.
INFO       2024-03-13 18:33:11,281 __main__ recognize 38  : Recognized person with id 3 and distance 15.350044250488281
INFO       2024-03-13 18:33:11,616 __main__ recognize 38  : Recognized person with id 3 and distance 14.763799667358398
INFO       2024-03-13 18:33:11,902 __main__ recognize 38  : Recognized person with id 3 and distance 14.396169662475586
INFO       2024-03-13 18:33:12,202 __main__ recognize 38  : Recognized person with id 3 and distance 14.545509338378906
INFO       2024-03-13 18:33:12,503 __main__ recognize 38  : Recognized person with id 3 and distance 14.149650573730469
INFO       2024-03-13 18:33:12,818 __main__ recognize 38  : Recognized person with id 3 and distance 16.310970306396484
INFO       2024-03-13 18:33:13,124 __main__ recognize 38  : Recognized person with id 3 and distance 15.234868049621582
INFO       2024-03-13 18:33:13,465 __main__ recognize 38  : Recognized person with id 3 and distance 15.41073989868164
INFO       2024-03-13 18:33:13,780 __main__ recognize 38  : Recognized person with id 3 and distance 14.935057640075684
INFO       2024-03-13 18:33:14,067 __main__ recognize 38  : Recognized person with id 3 and distance 16.420339584350586
INFO       2024-03-13 18:33:14,372 __main__ recognize 38  : Recognized person with id 3 and distance 15.808503150939941
INFO       2024-03-13 18:33:14,666 __main__ recognize 38  : Recognized person with id 3 and distance 16.367462158203125
INFO       2024-03-13 18:33:14,973 __main__ recognize 38  : Recognized person with id 3 and distance 15.24083137512207
INFO       2024-03-13 18:33:15,275 __main__ recognize 38  : Recognized person with id 3 and distance 15.036178588867188
INFO       2024-03-13 18:33:15,577 __main__ recognize 38  : Recognized person with id 3 and distance 15.696721076965332
INFO       2024-03-13 18:33:15,866 __main__ recognize 38  : Recognized person with id 3 and distance 16.277297973632812
INFO       2024-03-13 18:33:16,154 __main__ recognize 38  : Recognized person with id 3 and distance 14.824480056762695
INFO       2024-03-13 18:33:16,449 __main__ recognize 38  : Recognized person with id 3 and distance 13.951138496398926
INFO       2024-03-13 18:33:16,735 __main__ recognize 38  : Recognized person with id 3 and distance 16.157665252685547
INFO       2024-03-13 18:33:17,038 __main__ recognize 38  : Recognized person with id 3 and distance 15.566887855529785
INFO       2024-03-13 18:33:17,326 __main__ recognize 38  : Recognized person with id 3 and distance 16.9229793548584
INFO       2024-03-13 18:33:17,612 __main__ recognize 38  : Recognized person with id 3 and distance 16.442569732666016
INFO       2024-03-13 18:33:17,900 __main__ recognize 38  : Recognized person with id 3 and distance 20.3414363861084
INFO       2024-03-13 18:33:18,188 __main__ recognize 38  : Recognized person with id 3 and distance 16.885944366455078
INFO       2024-03-13 18:33:18,484 __main__ recognize 38  : Recognized person with id 3 and distance 16.64457130432129
INFO       2024-03-13 18:33:18,771 __main__ recognize 38  : Recognized person with id 3 and distance 18.201581954956055
INFO       2024-03-13 18:33:19,059 __main__ recognize 38  : Recognized person with id 3 and distance 17.23395347595215
INFO       2024-03-13 18:33:19,348 __main__ recognize 38  : Recognized person with id 3 and distance 16.98805046081543
INFO       2024-03-13 18:33:19,637 __main__ recognize 38  : Recognized person with id 3 and distance 18.72475242614746
INFO       2024-03-13 18:33:19,949 __main__ recognize 38  : Recognized person with id 3 and distance 18.239307403564453
INFO       2024-03-13 18:33:20,245 __main__ recognize 38  : Recognized person with id 3 and distance 15.519258499145508
INFO       2024-03-13 19:06:11,761 __main__ recognize 38  : Recognized person with id 0 and distance 22.562042236328125
INFO       2024-03-13 19:06:12,001 __main__ recognize 38  : Recognized person with id 3 and distance 19.60882568359375
INFO       2024-03-13 19:06:12,230 __main__ recognize 38  : Recognized person with id 3 and distance 18.513547897338867
INFO       2024-03-13 19:06:12,458 __main__ recognize 38  : Recognized person with id 3 and distance 21.29472541809082
INFO       2024-03-13 19:06:12,686 __main__ recognize 38  : Recognized person with id 3 and distance 16.019697189331055
INFO       2024-03-13 19:06:12,907 __main__ recognize 38  : Recognized person with id 3 and distance 12.877439498901367
INFO       2024-03-13 19:06:13,129 __main__ recognize 38  : Recognized person with id 3 and distance 12.169082641601562
INFO       2024-03-13 19:06:13,353 __main__ recognize 38  : Recognized person with id 3 and distance 13.578213691711426
INFO       2024-03-13 19:06:13,575 __main__ recognize 38  : Recognized person with id 3 and distance 14.386800765991211
INFO       2024-03-13 19:06:13,798 __main__ recognize 38  : Recognized person with id 3 and distance 11.520439147949219
INFO       2024-03-13 19:06:14,026 __main__ recognize 38  : Recognized person with id 3 and distance 14.15827751159668
INFO       2024-03-13 19:06:14,253 __main__ recognize 38  : Recognized person with id 3 and distance 11.190461158752441
INFO       2024-03-13 19:06:14,482 __main__ recognize 38  : Recognized person with id 3 and distance 13.357239723205566
INFO       2024-03-13 19:06:14,718 __main__ recognize 38  : Recognized person with id 3 and distance 12.409753799438477
INFO       2024-03-13 19:06:14,944 __main__ recognize 38  : Recognized person with id 3 and distance 11.190916061401367
INFO       2024-03-13 19:06:15,166 __main__ recognize 38  : Recognized person with id 3 and distance 13.248021125793457
INFO       2024-03-13 19:06:15,389 __main__ recognize 38  : Recognized person with id 3 and distance 12.872563362121582
INFO       2024-03-13 19:06:15,610 __main__ recognize 38  : Recognized person with id 3 and distance 14.032468795776367
INFO       2024-03-13 19:06:15,834 __main__ recognize 38  : Recognized person with id 3 and distance 13.183991432189941
INFO       2024-03-13 19:06:16,052 __main__ recognize 38  : Recognized person with id 3 and distance 11.08149528503418
INFO       2024-03-13 19:06:16,289 __main__ recognize 38  : Recognized person with id 3 and distance 12.159207344055176
INFO       2024-03-13 19:06:16,526 __main__ recognize 38  : Recognized person with id 3 and distance 12.132458686828613
INFO       2024-03-13 19:06:16,764 __main__ recognize 38  : Recognized person with id 3 and distance 12.030187606811523
INFO       2024-03-13 19:06:16,993 __main__ recognize 38  : Recognized person with id 3 and distance 11.448860168457031
INFO       2024-03-13 19:06:17,222 __main__ recognize 38  : Recognized person with id 3 and distance 13.341838836669922
INFO       2024-03-13 19:06:17,451 __main__ recognize 38  : Recognized person with id 3 and distance 10.969889640808105
INFO       2024-03-13 20:36:04,140 pika.adapters.blocking_connection close 802 : Closing connection (200): Normal shutdown
INFO       2024-03-13 20:36:04,142 pika.channel close 536 : Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7372be3d1fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
INFO       2024-03-13 20:36:04,142 pika.channel _on_closeok 1133: Received <Channel.CloseOk> on <Channel number=1 CLOSING conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7372be3d1fc0> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
INFO       2024-03-13 20:36:04,143 pika.connection close 1316: Closing connection (200): 'Normal shutdown'
INFO       2024-03-13 20:36:04,143 pika.adapters.utils.io_services_utils abort 731 : Aborting transport connection: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 34224), raddr=('127.0.0.1', 5672)>
INFO       2024-03-13 20:36:04,143 pika.adapters.utils.io_services_utils _initiate_abort 904 : _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 34224), raddr=('127.0.0.1', 5672)>
INFO       2024-03-13 20:36:04,144 pika.adapters.utils.io_services_utils _deactivate 869 : Deactivating transport: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 34224), raddr=('127.0.0.1', 5672)>
INFO       2024-03-13 20:36:04,144 pika.connection _on_stream_terminated 2017: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
INFO       2024-03-13 20:36:04,144 pika.connection _on_stream_terminated 2086: Stack terminated due to ConnectionClosedByClient: (200) 'Normal shutdown'
INFO       2024-03-13 20:36:04,144 pika.adapters.utils.io_services_utils _close_and_finalize 882 : Closing transport socket and unlinking: state=3; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 34224), raddr=('127.0.0.1', 5672)>
INFO       2024-03-13 20:36:04,144 pika.adapters.blocking_connection _flush_output 525 : User-initiated close: result=BlockingConnection__OnClosedArgs(connection=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>, error=ConnectionClosedByClient: (200) 'Normal shutdown')
INFO       2024-03-13 20:36:04,144 __main__ <module> 76  : Connection closed. Application closing.
INFO       2024-03-13 20:39:56,830 __main__ <module> 57  : Recognizer service is starting...
INFO       2024-03-13 20:39:56,831 pika.adapters.utils.connection_workflow start 179 : Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
INFO       2024-03-13 20:39:56,832 pika.adapters.utils.io_services_utils _on_writable 345 : Socket connected: <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 43886), raddr=('127.0.0.1', 5672)>
INFO       2024-03-13 20:39:56,832 pika.adapters.utils.connection_workflow _on_transport_establishment_done 428 : Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7725589de200>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7725589de200> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
INFO       2024-03-13 20:39:56,834 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 293 : AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7725589de200> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-13 20:39:56,834 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 725 : AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7725589de200> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-13 20:39:56,834 pika.adapters.blocking_connection _create_connection 453 : Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7725589de200> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-13 20:39:56,834 pika.adapters.blocking_connection __init__ 1264: Created channel=1
INFO       2024-03-13 20:40:01,841 __main__ <module> 72  : Connection established.
INFO       2024-03-13 20:44:23,750 __main__ recognize 39  : Recognized person with id 3 and distance 17.854875564575195
INFO       2024-03-13 20:44:23,985 __main__ recognize 39  : Recognized person with id 3 and distance 16.111873626708984
INFO       2024-03-13 20:44:24,223 __main__ recognize 39  : Recognized person with id 3 and distance 16.26494598388672
INFO       2024-03-13 20:44:24,450 __main__ recognize 39  : Recognized person with id 3 and distance 14.391570091247559
INFO       2024-03-13 20:44:24,679 __main__ recognize 39  : Recognized person with id 3 and distance 13.977867126464844
INFO       2024-03-13 20:44:24,902 __main__ recognize 39  : Recognized person with id 3 and distance 14.982490539550781
INFO       2024-03-13 20:44:25,124 __main__ recognize 39  : Recognized person with id 3 and distance 15.210615158081055
INFO       2024-03-13 20:44:25,347 __main__ recognize 39  : Recognized person with id 3 and distance 15.624043464660645
INFO       2024-03-13 20:44:25,570 __main__ recognize 39  : Recognized person with id 3 and distance 14.997772216796875
INFO       2024-03-13 20:44:25,792 __main__ recognize 39  : Recognized person with id 3 and distance 14.22799301147461
INFO       2024-03-13 20:44:26,013 __main__ recognize 39  : Recognized person with id 3 and distance 14.552713394165039
INFO       2024-03-13 20:44:26,235 __main__ recognize 39  : Recognized person with id 3 and distance 14.323882102966309
INFO       2024-03-13 20:44:26,457 __main__ recognize 39  : Recognized person with id 3 and distance 15.009099006652832
INFO       2024-03-13 20:44:26,677 __main__ recognize 39  : Recognized person with id 3 and distance 12.150553703308105
INFO       2024-03-13 20:44:26,900 __main__ recognize 39  : Recognized person with id 3 and distance 12.783276557922363
INFO       2024-03-13 20:44:27,121 __main__ recognize 39  : Recognized person with id 3 and distance 14.771156311035156
INFO       2024-03-13 20:44:27,345 __main__ recognize 39  : Recognized person with id 3 and distance 13.50946044921875
INFO       2024-03-13 20:44:27,566 __main__ recognize 39  : Recognized person with id 3 and distance 13.724387168884277
INFO       2024-03-13 20:44:27,790 __main__ recognize 39  : Recognized person with id 3 and distance 13.133390426635742
INFO       2024-03-13 20:44:28,019 __main__ recognize 39  : Recognized person with id 3 and distance 15.603347778320312
INFO       2024-03-13 20:46:11,073 __main__ recognize 39  : Recognized person with id 3 and distance 13.965774536132812
INFO       2024-03-13 20:46:11,317 __main__ recognize 39  : Recognized person with id 3 and distance 13.570340156555176
INFO       2024-03-13 20:46:11,550 __main__ recognize 39  : Recognized person with id 3 and distance 13.075701713562012
INFO       2024-03-13 20:46:11,785 __main__ recognize 39  : Recognized person with id 3 and distance 12.158226013183594
INFO       2024-03-13 20:46:12,010 __main__ recognize 39  : Recognized person with id 3 and distance 15.379415512084961
INFO       2024-03-13 20:46:12,245 __main__ recognize 39  : Recognized person with id 3 and distance 13.5728178024292
INFO       2024-03-13 20:46:12,471 __main__ recognize 39  : Recognized person with id 3 and distance 15.215585708618164
INFO       2024-03-13 20:46:12,701 __main__ recognize 39  : Recognized person with id 3 and distance 14.997802734375
INFO       2024-03-13 20:46:12,925 __main__ recognize 39  : Recognized person with id 3 and distance 12.32043743133545
INFO       2024-03-13 20:46:13,152 __main__ recognize 39  : Recognized person with id 3 and distance 11.869190216064453
INFO       2024-03-13 20:46:13,379 __main__ recognize 39  : Recognized person with id 3 and distance 15.417643547058105
INFO       2024-03-13 20:46:13,605 __main__ recognize 39  : Recognized person with id 3 and distance 14.969125747680664
INFO       2024-03-13 20:46:13,831 __main__ recognize 39  : Recognized person with id 3 and distance 13.127337455749512
INFO       2024-03-13 20:46:14,057 __main__ recognize 39  : Recognized person with id 3 and distance 12.426553726196289
INFO       2024-03-13 20:46:14,293 __main__ recognize 39  : Recognized person with id 3 and distance 14.64720630645752
INFO       2024-03-13 20:46:14,524 __main__ recognize 39  : Recognized person with id 3 and distance 15.490800857543945
INFO       2024-03-13 20:46:14,748 __main__ recognize 39  : Recognized person with id 3 and distance 12.204188346862793
INFO       2024-03-13 20:46:14,980 __main__ recognize 39  : Recognized person with id 3 and distance 15.652534484863281
INFO       2024-03-13 20:46:15,205 __main__ recognize 39  : Recognized person with id 3 and distance 15.485895156860352
INFO       2024-03-13 20:46:15,431 __main__ recognize 39  : Recognized person with id 3 and distance 15.089656829833984
INFO       2024-03-13 20:46:15,685 __main__ recognize 39  : Recognized person with id 3 and distance 14.030653953552246
INFO       2024-03-13 20:46:15,916 __main__ recognize 39  : Recognized person with id 3 and distance 13.872098922729492
INFO       2024-03-13 20:46:16,143 __main__ recognize 39  : Recognized person with id 3 and distance 14.546358108520508
INFO       2024-03-13 20:46:16,368 __main__ recognize 39  : Recognized person with id 3 and distance 13.9093599319458
INFO       2024-03-13 20:46:16,594 __main__ recognize 39  : Recognized person with id 3 and distance 13.859933853149414
INFO       2024-03-13 20:46:16,825 __main__ recognize 39  : Recognized person with id 3 and distance 14.150909423828125
INFO       2024-03-13 20:46:17,049 __main__ recognize 39  : Recognized person with id 3 and distance 14.646493911743164
INFO       2024-03-13 20:46:17,276 __main__ recognize 39  : Recognized person with id 3 and distance 15.103376388549805
INFO       2024-03-13 20:46:17,505 __main__ recognize 39  : Recognized person with id 3 and distance 12.72972297668457
INFO       2024-03-13 20:46:17,737 __main__ recognize 39  : Recognized person with id 3 and distance 14.944375038146973
INFO       2024-03-13 21:01:29,262 __main__ recognize 39  : Recognized person with id 3 and distance 14.289719581604004
INFO       2024-03-13 21:01:29,490 __main__ recognize 39  : Recognized person with id 3 and distance 13.90909481048584
INFO       2024-03-13 21:01:29,718 __main__ recognize 39  : Recognized person with id 3 and distance 14.380426406860352
INFO       2024-03-13 21:01:29,942 __main__ recognize 39  : Recognized person with id 3 and distance 14.665706634521484
INFO       2024-03-13 21:01:30,167 __main__ recognize 39  : Recognized person with id 3 and distance 16.188373565673828
INFO       2024-03-13 21:01:30,392 __main__ recognize 39  : Recognized person with id 3 and distance 14.4906644821167
INFO       2024-03-13 21:01:30,616 __main__ recognize 39  : Recognized person with id 3 and distance 17.219879150390625
INFO       2024-03-13 21:01:30,838 __main__ recognize 39  : Recognized person with id 3 and distance 17.94474983215332
INFO       2024-03-13 21:01:31,063 __main__ recognize 39  : Recognized person with id 3 and distance 17.609294891357422
INFO       2024-03-13 21:01:31,285 __main__ recognize 39  : Recognized person with id 3 and distance 15.011569023132324
INFO       2024-03-13 21:01:31,512 __main__ recognize 39  : Recognized person with id 3 and distance 17.39108657836914
INFO       2024-03-13 21:01:31,741 __main__ recognize 39  : Recognized person with id 3 and distance 16.610395431518555
INFO       2024-03-13 21:01:31,970 __main__ recognize 39  : Recognized person with id 3 and distance 17.051706314086914
INFO       2024-03-13 21:01:32,199 __main__ recognize 39  : Recognized person with id 3 and distance 16.319429397583008
INFO       2024-03-13 21:01:32,429 __main__ recognize 39  : Recognized person with id 3 and distance 15.504215240478516
INFO       2024-03-13 21:01:32,658 __main__ recognize 39  : Recognized person with id 3 and distance 16.079103469848633
INFO       2024-03-13 21:01:32,887 __main__ recognize 39  : Recognized person with id 3 and distance 14.390097618103027
INFO       2024-03-13 21:01:33,113 __main__ recognize 39  : Recognized person with id 3 and distance 17.233346939086914
INFO       2024-03-13 21:01:33,402 __main__ recognize 39  : Recognized person with id 3 and distance 15.927508354187012
INFO       2024-03-13 21:01:33,627 __main__ recognize 39  : Recognized person with id 3 and distance 15.128974914550781
INFO       2024-03-13 21:01:33,851 __main__ recognize 39  : Recognized person with id 3 and distance 13.392674446105957
INFO       2024-03-13 21:01:34,078 __main__ recognize 39  : Recognized person with id 3 and distance 15.063613891601562
INFO       2024-03-13 21:01:34,306 __main__ recognize 39  : Recognized person with id 3 and distance 13.80681037902832
INFO       2024-03-13 21:01:34,534 __main__ recognize 39  : Recognized person with id 3 and distance 16.202896118164062
INFO       2024-03-13 21:01:34,776 __main__ recognize 39  : Recognized person with id 3 and distance 12.71353816986084
INFO       2024-03-13 21:01:35,032 __main__ recognize 39  : Recognized person with id 3 and distance 13.703969955444336
INFO       2024-03-13 21:01:35,303 __main__ recognize 39  : Recognized person with id 3 and distance 13.568572998046875
INFO       2024-03-13 21:01:35,548 __main__ recognize 39  : Recognized person with id 3 and distance 13.971541404724121
INFO       2024-03-13 21:01:35,787 __main__ recognize 39  : Recognized person with id 3 and distance 14.3943452835083
INFO       2024-03-13 21:04:27,245 __main__ recognize 39  : Recognized person with id 3 and distance 13.186240196228027
INFO       2024-03-13 21:04:27,474 __main__ recognize 39  : Recognized person with id 3 and distance 12.957740783691406
INFO       2024-03-13 21:04:27,698 __main__ recognize 39  : Recognized person with id 3 and distance 12.583222389221191
INFO       2024-03-13 21:04:27,923 __main__ recognize 39  : Recognized person with id 3 and distance 12.624382019042969
INFO       2024-03-13 21:04:28,148 __main__ recognize 39  : Recognized person with id 3 and distance 12.290247917175293
INFO       2024-03-13 21:04:28,370 __main__ recognize 39  : Recognized person with id 3 and distance 15.063082695007324
INFO       2024-03-13 21:04:28,596 __main__ recognize 39  : Recognized person with id 3 and distance 13.509788513183594
INFO       2024-03-13 21:04:28,824 __main__ recognize 39  : Recognized person with id 3 and distance 13.956923484802246
INFO       2024-03-13 21:04:29,047 __main__ recognize 39  : Recognized person with id 3 and distance 11.868796348571777
INFO       2024-03-13 21:04:29,276 __main__ recognize 39  : Recognized person with id 3 and distance 12.75595474243164
INFO       2024-03-13 21:04:29,507 __main__ recognize 39  : Recognized person with id 3 and distance 12.429768562316895
INFO       2024-03-13 21:04:29,735 __main__ recognize 39  : Recognized person with id 3 and distance 12.75397777557373
INFO       2024-03-13 21:04:29,963 __main__ recognize 39  : Recognized person with id 3 and distance 12.19882869720459
INFO       2024-03-13 21:04:30,187 __main__ recognize 39  : Recognized person with id 3 and distance 13.327179908752441
INFO       2024-03-13 21:04:30,421 __main__ recognize 39  : Recognized person with id 3 and distance 12.542436599731445
INFO       2024-03-13 21:04:30,655 __main__ recognize 39  : Recognized person with id 3 and distance 14.213337898254395
INFO       2024-03-13 21:04:30,890 __main__ recognize 39  : Recognized person with id 3 and distance 13.22768783569336
INFO       2024-03-13 21:04:31,127 __main__ recognize 39  : Recognized person with id 3 and distance 14.822615623474121
INFO       2024-03-13 21:04:31,363 __main__ recognize 39  : Recognized person with id 3 and distance 13.689814567565918
INFO       2024-03-13 21:07:02,029 __main__ recognize 39  : Recognized person with id 3 and distance 12.167468070983887
INFO       2024-03-13 21:07:02,260 __main__ recognize 39  : Recognized person with id 3 and distance 13.618637084960938
INFO       2024-03-13 21:07:02,490 __main__ recognize 39  : Recognized person with id 3 and distance 12.864943504333496
INFO       2024-03-13 21:07:02,720 __main__ recognize 39  : Recognized person with id 3 and distance 11.274192810058594
INFO       2024-03-13 21:07:02,947 __main__ recognize 39  : Recognized person with id 3 and distance 16.446762084960938
INFO       2024-03-13 21:07:03,180 __main__ recognize 39  : Recognized person with id 3 and distance 14.039783477783203
INFO       2024-03-13 21:07:03,408 __main__ recognize 39  : Recognized person with id 3 and distance 11.912667274475098
INFO       2024-03-13 21:07:03,632 __main__ recognize 39  : Recognized person with id 3 and distance 12.371953964233398
INFO       2024-03-13 21:07:03,857 __main__ recognize 39  : Recognized person with id 3 and distance 12.499560356140137
INFO       2024-03-13 21:07:04,083 __main__ recognize 39  : Recognized person with id 3 and distance 12.540443420410156
INFO       2024-03-13 21:07:04,310 __main__ recognize 39  : Recognized person with id 3 and distance 12.778794288635254
INFO       2024-03-13 21:07:04,537 __main__ recognize 39  : Recognized person with id 3 and distance 12.521099090576172
INFO       2024-03-13 21:07:04,769 __main__ recognize 39  : Recognized person with id 3 and distance 13.601611137390137
INFO       2024-03-13 21:07:05,003 __main__ recognize 39  : Recognized person with id 3 and distance 13.629005432128906
INFO       2024-03-13 21:10:12,672 __main__ recognize 39  : Recognized person with id 3 and distance 14.04942798614502
INFO       2024-03-13 21:10:12,903 __main__ recognize 39  : Recognized person with id 3 and distance 15.00472640991211
INFO       2024-03-13 21:10:13,128 __main__ recognize 39  : Recognized person with id 3 and distance 14.239766120910645
INFO       2024-03-13 21:10:13,353 __main__ recognize 39  : Recognized person with id 3 and distance 13.592158317565918
INFO       2024-03-13 21:10:13,580 __main__ recognize 39  : Recognized person with id 3 and distance 14.160707473754883
INFO       2024-03-13 21:10:13,804 __main__ recognize 39  : Recognized person with id 3 and distance 15.223532676696777
INFO       2024-03-13 21:10:14,031 __main__ recognize 39  : Recognized person with id 3 and distance 15.750535011291504
INFO       2024-03-13 21:10:14,256 __main__ recognize 39  : Recognized person with id 3 and distance 14.778960227966309
INFO       2024-03-13 21:10:14,483 __main__ recognize 39  : Recognized person with id 3 and distance 17.567842483520508
INFO       2024-03-13 21:10:14,709 __main__ recognize 39  : Recognized person with id 3 and distance 17.714065551757812
INFO       2024-03-13 21:10:14,938 __main__ recognize 39  : Recognized person with id 3 and distance 15.701516151428223
INFO       2024-03-13 21:10:15,163 __main__ recognize 39  : Recognized person with id 3 and distance 18.145174026489258
INFO       2024-03-13 21:10:15,387 __main__ recognize 39  : Recognized person with id 3 and distance 14.860261917114258
INFO       2024-03-13 21:10:15,620 __main__ recognize 39  : Recognized person with id 3 and distance 18.02482795715332
INFO       2024-03-13 21:10:15,850 __main__ recognize 39  : Recognized person with id 3 and distance 15.924569129943848
INFO       2024-03-13 21:10:16,085 __main__ recognize 39  : Recognized person with id 3 and distance 15.540094375610352
INFO       2024-03-13 21:10:16,319 __main__ recognize 39  : Recognized person with id 3 and distance 17.755247116088867
INFO       2024-03-13 21:10:58,100 __main__ recognize 39  : Recognized person with id 3 and distance 13.828816413879395
INFO       2024-03-13 21:10:58,341 __main__ recognize 39  : Recognized person with id 3 and distance 13.624931335449219
INFO       2024-03-13 21:10:58,571 __main__ recognize 39  : Recognized person with id 3 and distance 14.022839546203613
INFO       2024-03-13 21:10:58,810 __main__ recognize 39  : Recognized person with id 3 and distance 13.690756797790527
INFO       2024-03-13 21:10:59,039 __main__ recognize 39  : Recognized person with id 3 and distance 13.752906799316406
INFO       2024-03-13 21:10:59,272 __main__ recognize 39  : Recognized person with id 3 and distance 14.62508487701416
INFO       2024-03-13 21:10:59,512 __main__ recognize 39  : Recognized person with id 3 and distance 13.82160758972168
INFO       2024-03-13 21:10:59,752 __main__ recognize 39  : Recognized person with id 3 and distance 14.137809753417969
INFO       2024-03-13 21:11:00,002 __main__ recognize 39  : Recognized person with id 3 and distance 15.58296012878418
INFO       2024-03-13 21:11:00,233 __main__ recognize 39  : Recognized person with id 3 and distance 15.202536582946777
INFO       2024-03-13 21:11:00,465 __main__ recognize 39  : Recognized person with id 3 and distance 16.860506057739258
INFO       2024-03-13 21:11:00,698 __main__ recognize 39  : Recognized person with id 3 and distance 14.815840721130371
INFO       2024-03-13 21:11:00,929 __main__ recognize 39  : Recognized person with id 3 and distance 14.83856201171875
INFO       2024-03-13 21:11:01,163 __main__ recognize 39  : Recognized person with id 3 and distance 15.234820365905762
INFO       2024-03-13 21:11:01,394 __main__ recognize 39  : Recognized person with id 3 and distance 15.004467964172363
INFO       2024-03-13 21:11:01,635 __main__ recognize 39  : Recognized person with id 3 and distance 14.716792106628418
INFO       2024-03-13 21:11:01,876 __main__ recognize 39  : Recognized person with id 3 and distance 14.350360870361328
INFO       2024-03-13 21:11:02,121 __main__ recognize 39  : Recognized person with id 3 and distance 15.065840721130371
INFO       2024-03-13 21:11:02,363 __main__ recognize 39  : Recognized person with id 3 and distance 14.621588706970215
INFO       2024-03-13 21:11:02,606 __main__ recognize 39  : Recognized person with id 3 and distance 14.05450439453125
INFO       2024-03-13 21:11:02,848 __main__ recognize 39  : Recognized person with id 3 and distance 14.373507499694824
INFO       2024-03-13 21:11:03,091 __main__ recognize 39  : Recognized person with id 3 and distance 14.417939186096191
INFO       2024-03-13 21:11:03,335 __main__ recognize 39  : Recognized person with id 3 and distance 13.093828201293945
INFO       2024-03-13 21:11:03,587 __main__ recognize 39  : Recognized person with id 3 and distance 15.971138954162598
INFO       2024-03-13 21:11:03,817 __main__ recognize 39  : Recognized person with id 3 and distance 15.382761001586914
INFO       2024-03-13 21:11:04,055 __main__ recognize 39  : Recognized person with id 3 and distance 14.442527770996094
INFO       2024-03-13 21:12:19,860 pika.adapters.blocking_connection close 802 : Closing connection (200): Normal shutdown
INFO       2024-03-13 21:12:19,862 pika.channel close 536 : Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7725589de200> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
INFO       2024-03-13 21:12:19,862 pika.channel _on_closeok 1133: Received <Channel.CloseOk> on <Channel number=1 CLOSING conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x7725589de200> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
INFO       2024-03-13 21:12:19,862 pika.connection close 1316: Closing connection (200): 'Normal shutdown'
INFO       2024-03-13 21:12:19,863 pika.adapters.utils.io_services_utils abort 731 : Aborting transport connection: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 43886), raddr=('127.0.0.1', 5672)>
INFO       2024-03-13 21:12:19,863 pika.adapters.utils.io_services_utils _initiate_abort 904 : _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 43886), raddr=('127.0.0.1', 5672)>
INFO       2024-03-13 21:12:19,863 pika.adapters.utils.io_services_utils _deactivate 869 : Deactivating transport: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 43886), raddr=('127.0.0.1', 5672)>
INFO       2024-03-13 21:12:19,863 pika.connection _on_stream_terminated 2017: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
INFO       2024-03-13 21:12:19,863 pika.connection _on_stream_terminated 2086: Stack terminated due to ConnectionClosedByClient: (200) 'Normal shutdown'
INFO       2024-03-13 21:12:19,864 pika.adapters.utils.io_services_utils _close_and_finalize 882 : Closing transport socket and unlinking: state=3; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 43886), raddr=('127.0.0.1', 5672)>
INFO       2024-03-13 21:12:19,864 pika.adapters.blocking_connection _flush_output 525 : User-initiated close: result=BlockingConnection__OnClosedArgs(connection=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>, error=ConnectionClosedByClient: (200) 'Normal shutdown')
INFO       2024-03-13 21:12:19,864 __main__ <module> 79  : Connection closed. Application closing.
INFO       2024-03-13 21:12:21,188 __main__ <module> 57  : Recognizer service is starting...
INFO       2024-03-13 21:12:21,190 pika.adapters.utils.connection_workflow start 179 : Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
INFO       2024-03-13 21:12:21,190 pika.adapters.utils.io_services_utils _on_writable 345 : Socket connected: <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 42510), raddr=('127.0.0.1', 5672)>
INFO       2024-03-13 21:12:21,190 pika.adapters.utils.connection_workflow _on_transport_establishment_done 428 : Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x74755ffd6200>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x74755ffd6200> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
INFO       2024-03-13 21:12:21,192 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 293 : AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x74755ffd6200> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-13 21:12:21,192 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 725 : AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x74755ffd6200> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-13 21:12:21,192 pika.adapters.blocking_connection _create_connection 453 : Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x74755ffd6200> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-13 21:12:21,192 pika.adapters.blocking_connection __init__ 1264: Created channel=1
INFO       2024-03-13 21:12:26,196 __main__ <module> 72  : Connection established.
INFO       2024-03-13 21:13:14,335 __main__ recognize 39  : Recognized person with id 3 and distance 12.273258209228516
INFO       2024-03-13 21:13:14,569 __main__ recognize 39  : Recognized person with id 3 and distance 13.275914192199707
INFO       2024-03-13 21:13:14,804 __main__ recognize 39  : Recognized person with id 3 and distance 13.445780754089355
INFO       2024-03-13 21:13:15,033 __main__ recognize 39  : Recognized person with id 3 and distance 13.514477729797363
INFO       2024-03-13 21:13:15,262 __main__ recognize 39  : Recognized person with id 3 and distance 15.504006385803223
INFO       2024-03-13 21:13:15,488 __main__ recognize 39  : Recognized person with id 3 and distance 12.063944816589355
INFO       2024-03-13 21:13:15,714 __main__ recognize 39  : Recognized person with id 3 and distance 11.073390007019043
INFO       2024-03-13 21:13:15,942 __main__ recognize 39  : Recognized person with id 3 and distance 11.266860008239746
INFO       2024-03-13 21:13:16,169 __main__ recognize 39  : Recognized person with id 3 and distance 10.984079360961914
INFO       2024-03-13 21:13:16,395 __main__ recognize 39  : Recognized person with id 3 and distance 12.424996376037598
INFO       2024-03-13 21:13:16,622 __main__ recognize 39  : Recognized person with id 3 and distance 11.17020320892334
INFO       2024-03-13 21:13:16,847 __main__ recognize 39  : Recognized person with id 3 and distance 10.837559700012207
INFO       2024-03-13 21:13:17,073 __main__ recognize 39  : Recognized person with id 3 and distance 11.528037071228027
INFO       2024-03-13 21:13:17,298 __main__ recognize 39  : Recognized person with id 3 and distance 11.172207832336426
INFO       2024-03-13 21:13:17,535 __main__ recognize 39  : Recognized person with id 3 and distance 12.427300453186035
INFO       2024-03-13 21:13:17,773 __main__ recognize 39  : Recognized person with id 3 and distance 11.278908729553223
INFO       2024-03-13 21:18:00,862 pika.adapters.blocking_connection close 802 : Closing connection (200): Normal shutdown
INFO       2024-03-13 21:18:00,862 pika.channel close 536 : Closing channel (200): 'Normal shutdown' on <Channel number=1 OPEN conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x74755ffd6200> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
INFO       2024-03-13 21:18:00,863 pika.channel _on_closeok 1133: Received <Channel.CloseOk> on <Channel number=1 CLOSING conn=<SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x74755ffd6200> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>>
INFO       2024-03-13 21:18:00,863 pika.connection close 1316: Closing connection (200): 'Normal shutdown'
INFO       2024-03-13 21:18:00,863 pika.adapters.utils.io_services_utils abort 731 : Aborting transport connection: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 42510), raddr=('127.0.0.1', 5672)>
INFO       2024-03-13 21:18:00,863 pika.adapters.utils.io_services_utils _initiate_abort 904 : _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 42510), raddr=('127.0.0.1', 5672)>
INFO       2024-03-13 21:18:00,863 pika.adapters.utils.io_services_utils _deactivate 869 : Deactivating transport: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 42510), raddr=('127.0.0.1', 5672)>
INFO       2024-03-13 21:18:00,863 pika.connection _on_stream_terminated 2017: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByClient: (200) 'Normal shutdown'
INFO       2024-03-13 21:18:00,863 pika.connection _on_stream_terminated 2086: Stack terminated due to ConnectionClosedByClient: (200) 'Normal shutdown'
INFO       2024-03-13 21:18:00,863 pika.adapters.utils.io_services_utils _close_and_finalize 882 : Closing transport socket and unlinking: state=3; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 42510), raddr=('127.0.0.1', 5672)>
INFO       2024-03-13 21:18:00,863 pika.adapters.blocking_connection _flush_output 525 : User-initiated close: result=BlockingConnection__OnClosedArgs(connection=<SelectConnection CLOSED transport=None params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>, error=ConnectionClosedByClient: (200) 'Normal shutdown')
INFO       2024-03-13 21:18:00,863 __main__ <module> 79  : Connection closed. Application closing.
INFO       2024-03-19 19:22:47,045 __main__ <module> 68  : Recognizer service is starting...
INFO       2024-03-19 19:22:47,049 pika.adapters.utils.connection_workflow start 179 : Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
INFO       2024-03-19 19:22:47,049 pika.adapters.utils.io_services_utils _on_writable 345 : Socket connected: <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 42476), raddr=('127.0.0.1', 5672)>
INFO       2024-03-19 19:22:47,050 pika.adapters.utils.connection_workflow _on_transport_establishment_done 428 : Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x74827a2d5f00>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x74827a2d5f00> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
INFO       2024-03-19 19:22:47,054 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 293 : AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x74827a2d5f00> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-19 19:22:47,054 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 725 : AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x74827a2d5f00> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-19 19:22:47,055 pika.adapters.blocking_connection _create_connection 453 : Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x74827a2d5f00> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-19 19:22:47,055 pika.adapters.blocking_connection __init__ 1264: Created channel=1
INFO       2024-03-19 19:22:52,075 __main__ <module> 85  : Connection established.
INFO       2024-03-19 19:31:27,727 __main__ <module> 68  : Recognizer service is starting...
INFO       2024-03-19 19:31:27,729 pika.adapters.utils.connection_workflow start 179 : Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
INFO       2024-03-19 19:31:27,729 pika.adapters.utils.io_services_utils _on_writable 345 : Socket connected: <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 40692), raddr=('127.0.0.1', 5672)>
INFO       2024-03-19 19:31:27,729 pika.adapters.utils.connection_workflow _on_transport_establishment_done 428 : Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x79b47c1d9f00>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x79b47c1d9f00> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
INFO       2024-03-19 19:31:27,733 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 293 : AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x79b47c1d9f00> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-19 19:31:27,733 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 725 : AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x79b47c1d9f00> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-19 19:31:27,733 pika.adapters.blocking_connection _create_connection 453 : Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x79b47c1d9f00> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-03-19 19:31:27,733 pika.adapters.blocking_connection __init__ 1264: Created channel=1
INFO       2024-03-19 19:31:32,739 __main__ <module> 85  : Connection established.
INFO       2024-03-19 19:33:05,255 __main__ recognize 47  : Recognized person with id 0 and distance 22.818004608154297
INFO       2024-03-19 19:33:05,503 __main__ recognize 47  : Recognized person with id 0 and distance 22.50346565246582
INFO       2024-03-19 19:33:05,748 __main__ recognize 47  : Recognized person with id 0 and distance 22.457963943481445
INFO       2024-03-19 19:33:06,004 __main__ recognize 47  : Recognized person with id 0 and distance 22.463747024536133
INFO       2024-03-19 19:33:06,261 __main__ recognize 47  : Recognized person with id 0 and distance 22.41929817199707
INFO       2024-03-19 19:33:06,506 __main__ recognize 47  : Recognized person with id 3 and distance 22.947885513305664
INFO       2024-03-19 19:33:06,748 __main__ recognize 47  : Recognized person with id 3 and distance 18.114282608032227
INFO       2024-03-19 19:33:06,988 __main__ recognize 47  : Recognized person with id 3 and distance 19.569137573242188
INFO       2024-03-19 19:33:07,230 __main__ recognize 47  : Recognized person with id 3 and distance 19.295318603515625
INFO       2024-03-19 19:33:07,473 __main__ recognize 47  : Recognized person with id 3 and distance 20.246313095092773
INFO       2024-03-19 19:33:07,722 __main__ recognize 47  : Recognized person with id 3 and distance 22.87506103515625
INFO       2024-03-19 19:33:07,971 __main__ recognize 47  : Recognized person with id 3 and distance 19.017051696777344
INFO       2024-03-19 19:33:08,213 __main__ recognize 47  : Recognized person with id 3 and distance 21.67439079284668
INFO       2024-03-19 19:33:08,459 __main__ recognize 47  : Recognized person with id 3 and distance 20.386430740356445
INFO       2024-03-19 19:33:08,707 __main__ recognize 47  : Recognized person with id 3 and distance 19.502416610717773
INFO       2024-03-19 19:33:08,953 __main__ recognize 47  : Recognized person with id 3 and distance 18.299739837646484
INFO       2024-03-19 19:33:09,193 __main__ recognize 47  : Recognized person with id 3 and distance 20.93065643310547
INFO       2024-03-19 19:33:09,437 __main__ recognize 47  : Recognized person with id 3 and distance 20.86293601989746
INFO       2024-03-19 19:33:09,684 __main__ recognize 47  : Recognized person with id 3 and distance 19.164541244506836
INFO       2024-03-19 19:33:09,931 __main__ recognize 47  : Recognized person with id 3 and distance 17.5930233001709
INFO       2024-03-19 19:33:10,182 __main__ recognize 47  : Recognized person with id 3 and distance 18.0582275390625
INFO       2024-03-19 19:33:10,432 __main__ recognize 47  : Recognized person with id 3 and distance 24.55498695373535
INFO       2024-03-19 19:33:10,668 __main__ recognize 47  : Recognized person with id 0 and distance 22.752328872680664
INFO       2024-03-19 19:33:10,908 __main__ recognize 47  : Recognized person with id 3 and distance 21.666046142578125
INFO       2024-03-19 19:33:11,146 __main__ recognize 47  : Recognized person with id 3 and distance 22.21552848815918
INFO       2024-03-19 19:33:11,384 __main__ recognize 47  : Recognized person with id 3 and distance 21.176776885986328
INFO       2024-03-19 19:33:11,624 __main__ recognize 47  : Recognized person with id 3 and distance 20.761085510253906
INFO       2024-03-19 19:33:11,863 __main__ recognize 47  : Recognized person with id 3 and distance 21.332752227783203
INFO       2024-03-19 19:33:12,104 __main__ recognize 47  : Recognized person with id 3 and distance 20.220033645629883
INFO       2024-03-19 19:33:12,331 __main__ recognize 47  : Recognized person with id 3 and distance 19.455095291137695
INFO       2024-03-19 19:33:12,572 __main__ recognize 47  : Recognized person with id 3 and distance 19.721378326416016
INFO       2024-03-19 19:33:12,812 __main__ recognize 47  : Recognized person with id 3 and distance 20.497854232788086
INFO       2024-03-19 19:33:13,054 __main__ recognize 47  : Recognized person with id 3 and distance 20.971973419189453
INFO       2024-03-19 19:33:13,293 __main__ recognize 47  : Recognized person with id 3 and distance 19.346988677978516
INFO       2024-03-19 19:33:13,534 __main__ recognize 47  : Recognized person with id 3 and distance 19.936138153076172
INFO       2024-03-19 19:33:13,773 __main__ recognize 47  : Recognized person with id 3 and distance 20.893468856811523
INFO       2024-03-19 19:33:14,011 __main__ recognize 47  : Recognized person with id 3 and distance 18.808879852294922
INFO       2024-03-19 19:33:14,257 __main__ recognize 47  : Recognized person with id 3 and distance 20.456132888793945
INFO       2024-03-19 19:33:14,512 __main__ recognize 47  : Recognized person with id 3 and distance 19.906513214111328
INFO       2024-03-19 19:33:14,768 __main__ recognize 47  : Recognized person with id 3 and distance 19.93376350402832
INFO       2024-03-19 19:33:15,024 __main__ recognize 47  : Recognized person with id 3 and distance 18.848543167114258
INFO       2024-04-17 14:55:23,872 __main__ <module> 68  : Recognizer service is starting...
INFO       2024-04-17 14:55:23,878 pika.adapters.utils.connection_workflow start 179 : Pika version 1.3.2 connecting to ('127.0.0.1', 5672)
INFO       2024-04-17 14:55:23,879 pika.adapters.utils.io_services_utils _on_writable 345 : Socket connected: <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 52950), raddr=('127.0.0.1', 5672)>
INFO       2024-04-17 14:55:23,880 pika.adapters.utils.connection_workflow _on_transport_establishment_done 428 : Streaming transport linked up: (<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76a8e94cdf30>, _StreamingProtocolShim: <SelectConnection PROTOCOL transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76a8e94cdf30> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>).
INFO       2024-04-17 14:55:23,884 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 293 : AMQPConnector - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76a8e94cdf30> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-04-17 14:55:23,885 pika.adapters.utils.connection_workflow _report_completion_and_cleanup 725 : AMQPConnectionWorkflow - reporting success: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76a8e94cdf30> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-04-17 14:55:23,885 pika.adapters.blocking_connection _create_connection 453 : Connection workflow succeeded: <SelectConnection OPEN transport=<pika.adapters.utils.io_services_utils._AsyncPlaintextTransport object at 0x76a8e94cdf30> params=<ConnectionParameters host=localhost port=5672 virtual_host=/ ssl=False>>
INFO       2024-04-17 14:55:23,885 pika.adapters.blocking_connection __init__ 1264: Created channel=1
INFO       2024-04-17 14:55:28,901 __main__ <module> 85  : Connection established.
INFO       2024-04-17 14:59:43,206 __main__ recognize 47  : Recognized person with id 0 and distance 22.359159469604492
INFO       2024-04-17 14:59:43,465 __main__ recognize 47  : Recognized person with id 0 and distance 22.52400779724121
INFO       2024-04-17 14:59:43,717 __main__ recognize 47  : Recognized person with id 0 and distance 22.54676628112793
INFO       2024-04-17 14:59:43,970 __main__ recognize 47  : Recognized person with id 0 and distance 22.461496353149414
INFO       2024-04-17 14:59:44,216 __main__ recognize 47  : Recognized person with id 0 and distance 22.59062957763672
INFO       2024-04-17 14:59:44,456 __main__ recognize 47  : Recognized person with id 3 and distance 21.679473876953125
INFO       2024-04-17 14:59:44,711 __main__ recognize 47  : Recognized person with id 3 and distance 21.818267822265625
INFO       2024-04-17 14:59:44,971 __main__ recognize 47  : Recognized person with id 3 and distance 20.401931762695312
INFO       2024-04-17 14:59:45,206 __main__ recognize 47  : Recognized person with id 3 and distance 21.454837799072266
INFO       2024-04-17 14:59:45,460 __main__ recognize 47  : Recognized person with id 3 and distance 21.764551162719727
INFO       2024-04-17 14:59:45,696 __main__ recognize 47  : Recognized person with id 3 and distance 22.885114669799805
INFO       2024-04-17 14:59:45,943 __main__ recognize 47  : Recognized person with id 3 and distance 22.122554779052734
INFO       2024-04-17 14:59:46,191 __main__ recognize 47  : Recognized person with id 3 and distance 23.90233612060547
INFO       2024-04-17 14:59:46,441 __main__ recognize 47  : Recognized person with id 0 and distance 22.423505783081055
INFO       2024-04-17 14:59:46,685 __main__ recognize 47  : Recognized person with id 0 and distance 22.171245574951172
INFO       2024-04-17 14:59:46,935 __main__ recognize 47  : Recognized person with id 3 and distance 19.995948791503906
INFO       2024-04-17 14:59:47,182 __main__ recognize 47  : Recognized person with id 3 and distance 22.604488372802734
INFO       2024-04-17 14:59:47,436 __main__ recognize 47  : Recognized person with id 3 and distance 22.68301773071289
INFO       2024-04-17 14:59:47,684 __main__ recognize 47  : Recognized person with id 3 and distance 23.39890480041504
INFO       2024-04-17 14:59:47,925 __main__ recognize 47  : Recognized person with id 3 and distance 22.743074417114258
INFO       2024-04-17 14:59:48,174 __main__ recognize 47  : Recognized person with id 3 and distance 22.740873336791992
INFO       2024-04-17 14:59:48,425 __main__ recognize 47  : Recognized person with id 3 and distance 21.58321762084961
INFO       2024-04-17 14:59:48,677 __main__ recognize 47  : Recognized person with id 3 and distance 22.103580474853516
INFO       2024-04-17 14:59:48,925 __main__ recognize 47  : Recognized person with id 3 and distance 21.44219207763672
INFO       2024-04-17 14:59:49,166 __main__ recognize 47  : Recognized person with id 3 and distance 20.870697021484375
INFO       2024-04-17 14:59:49,423 __main__ recognize 47  : Recognized person with id 3 and distance 22.791414260864258
INFO       2024-04-17 14:59:49,664 __main__ recognize 47  : Recognized person with id 3 and distance 21.004634857177734
INFO       2024-04-17 14:59:49,903 __main__ recognize 47  : Recognized person with id 3 and distance 22.71990203857422
INFO       2024-04-17 14:59:50,139 __main__ recognize 47  : Recognized person with id 3 and distance 19.884130477905273
INFO       2024-04-17 14:59:50,379 __main__ recognize 47  : Recognized person with id 3 and distance 20.80514907836914
INFO       2024-04-17 14:59:50,635 __main__ recognize 47  : Recognized person with id 3 and distance 20.64179039001465
INFO       2024-04-17 14:59:50,885 __main__ recognize 47  : Recognized person with id 3 and distance 20.619001388549805
INFO       2024-04-17 14:59:51,144 __main__ recognize 47  : Recognized person with id 3 and distance 20.662206649780273
INFO       2024-04-17 14:59:51,379 __main__ recognize 47  : Recognized person with id 3 and distance 20.81501579284668
INFO       2024-04-17 14:59:51,624 __main__ recognize 47  : Recognized person with id 3 and distance 21.670759201049805
INFO       2024-04-17 14:59:51,861 __main__ recognize 47  : Recognized person with id 3 and distance 22.499460220336914
INFO       2024-04-17 14:59:52,099 __main__ recognize 47  : Recognized person with id 3 and distance 23.405616760253906
INFO       2024-04-17 14:59:52,349 __main__ recognize 47  : Recognized person with id 3 and distance 23.139978408813477
INFO       2024-04-17 14:59:52,595 __main__ recognize 47  : Recognized person with id 3 and distance 21.80358123779297
INFO       2024-04-17 14:59:52,851 __main__ recognize 47  : Recognized person with id 3 and distance 23.09999656677246
INFO       2024-04-17 14:59:53,099 __main__ recognize 47  : Recognized person with id 3 and distance 24.161401748657227
INFO       2024-04-17 14:59:53,352 __main__ recognize 47  : Recognized person with id 3 and distance 22.270782470703125
INFO       2024-04-17 14:59:53,584 __main__ recognize 47  : Recognized person with id 3 and distance 22.32208251953125
INFO       2024-04-17 14:59:53,831 __main__ recognize 47  : Recognized person with id 3 and distance 23.21142578125
INFO       2024-04-17 14:59:54,092 __main__ recognize 47  : Recognized person with id 3 and distance 23.964397430419922
INFO       2024-04-17 14:59:54,330 __main__ recognize 47  : Recognized person with id 3 and distance 23.26426124572754
INFO       2024-04-17 14:59:54,569 __main__ recognize 47  : Recognized person with id 3 and distance 23.127893447875977
INFO       2024-04-17 14:59:54,820 __main__ recognize 47  : Recognized person with id 3 and distance 22.12859535217285
INFO       2024-04-17 14:59:55,066 __main__ recognize 47  : Recognized person with id 3 and distance 21.434772491455078
INFO       2024-04-17 14:59:55,319 __main__ recognize 47  : Recognized person with id 3 and distance 22.213748931884766
INFO       2024-04-17 14:59:55,585 __main__ recognize 47  : Recognized person with id 3 and distance 21.197669982910156
INFO       2024-04-17 14:59:55,833 __main__ recognize 47  : Recognized person with id 3 and distance 23.763193130493164
INFO       2024-04-17 14:59:56,093 __main__ recognize 47  : Recognized person with id 3 and distance 22.519222259521484
INFO       2024-04-17 14:59:56,337 __main__ recognize 47  : Recognized person with id 3 and distance 23.18564224243164
INFO       2024-04-17 14:59:56,578 __main__ recognize 47  : Recognized person with id 3 and distance 22.2816104888916
INFO       2024-04-17 14:59:56,836 __main__ recognize 47  : Recognized person with id 3 and distance 23.313919067382812
INFO       2024-04-17 14:59:57,076 __main__ recognize 47  : Recognized person with id 3 and distance 20.57530975341797
INFO       2024-04-17 14:59:57,319 __main__ recognize 47  : Recognized person with id 3 and distance 20.87506866455078
INFO       2024-04-17 14:59:57,575 __main__ recognize 47  : Recognized person with id 3 and distance 21.361751556396484
INFO       2024-04-17 14:59:57,828 __main__ recognize 47  : Recognized person with id 3 and distance 21.731380462646484
INFO       2024-04-17 14:59:58,066 __main__ recognize 47  : Recognized person with id 3 and distance 21.754499435424805
INFO       2024-04-17 15:02:23,349 __main__ recognize 47  : Recognized person with id 3 and distance 22.277620315551758
INFO       2024-04-17 15:02:23,588 __main__ recognize 47  : Recognized person with id 3 and distance 21.128042221069336
INFO       2024-04-17 15:02:23,819 __main__ recognize 47  : Recognized person with id 3 and distance 24.075977325439453
INFO       2024-04-17 15:02:24,045 __main__ recognize 47  : Recognized person with id 3 and distance 24.550931930541992
INFO       2024-04-17 15:02:24,273 __main__ recognize 47  : Recognized person with id 3 and distance 22.13703155517578
INFO       2024-04-17 15:02:24,503 __main__ recognize 47  : Recognized person with id 3 and distance 22.247095108032227
INFO       2024-04-17 15:02:24,729 __main__ recognize 47  : Recognized person with id 3 and distance 23.025440216064453
INFO       2024-04-17 15:02:24,967 __main__ recognize 47  : Recognized person with id 3 and distance 23.078975677490234
INFO       2024-04-17 15:02:25,192 __main__ recognize 47  : Recognized person with id 3 and distance 22.785823822021484
INFO       2024-04-17 15:02:25,424 __main__ recognize 47  : Recognized person with id 3 and distance 22.066070556640625
INFO       2024-04-17 15:02:25,651 __main__ recognize 47  : Recognized person with id 3 and distance 23.039243698120117
INFO       2024-04-17 15:02:25,879 __main__ recognize 47  : Recognized person with id 3 and distance 23.360462188720703
INFO       2024-04-17 15:02:26,114 __main__ recognize 47  : Recognized person with id 3 and distance 21.503292083740234
INFO       2024-04-17 15:02:26,345 __main__ recognize 47  : Recognized person with id 3 and distance 17.510311126708984
INFO       2024-04-17 15:02:26,567 __main__ recognize 47  : Recognized person with id 3 and distance 16.556848526000977
INFO       2024-04-17 15:02:26,793 __main__ recognize 47  : Recognized person with id 3 and distance 17.83201789855957
INFO       2024-04-17 15:02:27,025 __main__ recognize 47  : Recognized person with id 3 and distance 16.365306854248047
INFO       2024-04-17 15:02:27,255 __main__ recognize 47  : Recognized person with id 3 and distance 17.345699310302734
INFO       2024-04-17 15:02:27,493 __main__ recognize 47  : Recognized person with id 3 and distance 17.294391632080078
INFO       2024-04-17 15:02:27,723 __main__ recognize 47  : Recognized person with id 3 and distance 18.095294952392578
INFO       2024-04-17 15:02:27,947 __main__ recognize 47  : Recognized person with id 3 and distance 18.218599319458008
INFO       2024-04-17 15:02:28,169 __main__ recognize 47  : Recognized person with id 3 and distance 16.997711181640625
INFO       2024-04-17 15:02:28,392 __main__ recognize 47  : Recognized person with id 3 and distance 18.97878646850586
INFO       2024-04-17 15:02:28,624 __main__ recognize 47  : Recognized person with id 3 and distance 17.731971740722656
INFO       2024-04-17 15:02:28,853 __main__ recognize 47  : Recognized person with id 3 and distance 17.70026969909668
INFO       2024-04-17 15:02:29,084 __main__ recognize 47  : Recognized person with id 3 and distance 18.439128875732422
INFO       2024-04-17 15:02:29,313 __main__ recognize 47  : Recognized person with id 3 and distance 18.338979721069336
INFO       2024-04-17 15:02:29,545 __main__ recognize 47  : Recognized person with id 3 and distance 17.443208694458008
INFO       2024-04-17 15:02:29,775 __main__ recognize 47  : Recognized person with id 3 and distance 18.38289451599121
INFO       2024-04-17 15:02:30,004 __main__ recognize 47  : Recognized person with id 3 and distance 17.122190475463867
INFO       2024-04-17 15:02:30,233 __main__ recognize 47  : Recognized person with id 3 and distance 18.340656280517578
INFO       2024-04-17 15:02:30,459 __main__ recognize 47  : Recognized person with id 3 and distance 18.521249771118164
INFO       2024-04-17 15:02:30,685 __main__ recognize 47  : Recognized person with id 3 and distance 17.84333610534668
INFO       2024-04-17 15:02:30,918 __main__ recognize 47  : Recognized person with id 3 and distance 18.830175399780273
INFO       2024-04-17 15:02:31,152 __main__ recognize 47  : Recognized person with id 3 and distance 17.88463592529297
INFO       2024-04-17 15:02:31,388 __main__ recognize 47  : Recognized person with id 3 and distance 18.01900863647461
INFO       2024-04-17 15:02:31,624 __main__ recognize 47  : Recognized person with id 3 and distance 18.702295303344727
INFO       2024-04-17 15:02:31,861 __main__ recognize 47  : Recognized person with id 3 and distance 16.48048210144043
INFO       2024-04-17 15:02:32,099 __main__ recognize 47  : Recognized person with id 3 and distance 17.97283935546875
INFO       2024-04-17 15:02:32,330 __main__ recognize 47  : Recognized person with id 3 and distance 17.480024337768555
INFO       2024-04-17 15:02:32,569 __main__ recognize 47  : Recognized person with id 3 and distance 16.682205200195312
INFO       2024-04-17 15:02:32,806 __main__ recognize 47  : Recognized person with id 3 and distance 18.355350494384766
INFO       2024-04-17 15:02:33,035 __main__ recognize 47  : Recognized person with id 3 and distance 19.857955932617188
INFO       2024-04-17 15:02:33,268 __main__ recognize 47  : Recognized person with id 3 and distance 18.66963768005371
INFO       2024-04-17 15:02:33,501 __main__ recognize 47  : Recognized person with id 3 and distance 19.116979598999023
INFO       2024-04-17 15:02:33,735 __main__ recognize 47  : Recognized person with id 3 and distance 18.23517608642578
INFO       2024-04-17 15:02:33,971 __main__ recognize 47  : Recognized person with id 3 and distance 18.401901245117188
INFO       2024-04-17 15:02:34,207 __main__ recognize 47  : Recognized person with id 3 and distance 18.65962028503418
INFO       2024-04-17 15:02:34,441 __main__ recognize 47  : Recognized person with id 3 and distance 18.150602340698242
INFO       2024-04-17 15:02:34,673 __main__ recognize 47  : Recognized person with id 3 and distance 18.188922882080078
INFO       2024-04-17 15:02:34,909 __main__ recognize 47  : Recognized person with id 3 and distance 17.836524963378906
INFO       2024-04-17 15:02:35,146 __main__ recognize 47  : Recognized person with id 3 and distance 19.167001724243164
INFO       2024-04-17 15:02:35,380 __main__ recognize 47  : Recognized person with id 3 and distance 18.704326629638672
INFO       2024-04-17 15:02:35,615 __main__ recognize 47  : Recognized person with id 3 and distance 16.706825256347656
INFO       2024-04-17 15:02:35,850 __main__ recognize 47  : Recognized person with id 3 and distance 18.05051612854004
INFO       2024-04-17 15:02:36,087 __main__ recognize 47  : Recognized person with id 3 and distance 16.982242584228516
INFO       2024-04-17 15:02:36,324 __main__ recognize 47  : Recognized person with id 3 and distance 19.44588851928711
INFO       2024-04-17 15:02:36,558 __main__ recognize 47  : Recognized person with id 3 and distance 18.955589294433594
INFO       2024-04-17 15:02:36,785 __main__ recognize 47  : Recognized person with id 3 and distance 18.049028396606445
INFO       2024-04-17 15:02:37,009 __main__ recognize 47  : Recognized person with id 3 and distance 18.17180061340332
INFO       2024-04-17 15:02:37,243 __main__ recognize 47  : Recognized person with id 3 and distance 19.10275650024414
INFO       2024-04-17 15:02:37,480 __main__ recognize 47  : Recognized person with id 3 and distance 18.49773597717285
INFO       2024-04-17 15:02:37,713 __main__ recognize 47  : Recognized person with id 3 and distance 18.06303596496582
INFO       2024-04-17 15:02:37,945 __main__ recognize 47  : Recognized person with id 3 and distance 19.099220275878906
INFO       2024-04-17 15:02:38,180 __main__ recognize 47  : Recognized person with id 3 and distance 19.01213264465332
INFO       2024-04-17 15:02:38,406 __main__ recognize 47  : Recognized person with id 3 and distance 18.3248233795166
INFO       2024-04-17 15:02:38,640 __main__ recognize 47  : Recognized person with id 3 and distance 23.01849365234375
INFO       2024-04-17 15:02:38,863 __main__ recognize 47  : Recognized person with id 3 and distance 19.71553611755371
INFO       2024-04-17 15:02:39,090 __main__ recognize 47  : Recognized person with id 3 and distance 21.364139556884766
INFO       2024-04-17 15:02:39,314 __main__ recognize 47  : Recognized person with id 3 and distance 21.47010040283203
INFO       2024-04-17 15:02:39,542 __main__ recognize 47  : Recognized person with id 3 and distance 18.476181030273438
INFO       2024-04-17 15:02:39,768 __main__ recognize 47  : Recognized person with id 3 and distance 20.95839500427246
INFO       2024-04-17 15:02:39,996 __main__ recognize 47  : Recognized person with id 3 and distance 20.129192352294922
INFO       2024-04-17 15:02:40,224 __main__ recognize 47  : Recognized person with id 3 and distance 20.646316528320312
INFO       2024-04-17 15:02:40,452 __main__ recognize 47  : Recognized person with id 3 and distance 17.246246337890625
INFO       2024-04-17 15:02:40,677 __main__ recognize 47  : Recognized person with id 3 and distance 19.351943969726562
INFO       2024-04-17 15:02:40,905 __main__ recognize 47  : Recognized person with id 3 and distance 19.579631805419922
INFO       2024-04-17 15:02:41,150 __main__ recognize 47  : Recognized person with id 3 and distance 19.597440719604492
INFO       2024-04-17 15:02:41,386 __main__ recognize 47  : Recognized person with id 3 and distance 19.310585021972656
INFO       2024-04-17 15:02:41,614 __main__ recognize 47  : Recognized person with id 3 and distance 19.839078903198242
INFO       2024-04-17 15:02:41,850 __main__ recognize 47  : Recognized person with id 3 and distance 19.88503074645996
INFO       2024-04-17 15:02:42,081 __main__ recognize 47  : Recognized person with id 3 and distance 21.60841941833496
INFO       2024-04-17 15:02:42,318 __main__ recognize 47  : Recognized person with id 3 and distance 18.522308349609375
INFO       2024-04-17 15:02:42,553 __main__ recognize 47  : Recognized person with id 3 and distance 20.18857765197754
INFO       2024-04-17 15:02:42,782 __main__ recognize 47  : Recognized person with id 3 and distance 20.39617156982422
INFO       2024-04-17 18:55:50,534 __main__ recognize 47  : Recognized person with id 3 and distance 19.06566619873047
INFO       2024-04-17 18:55:50,788 __main__ recognize 47  : Recognized person with id 3 and distance 17.23107147216797
INFO       2024-04-17 18:55:51,037 __main__ recognize 47  : Recognized person with id 3 and distance 18.72121238708496
INFO       2024-04-17 18:55:51,304 __main__ recognize 47  : Recognized person with id 3 and distance 18.621490478515625
INFO       2024-04-17 18:55:51,552 __main__ recognize 47  : Recognized person with id 3 and distance 19.29133415222168
INFO       2024-04-17 18:55:51,794 __main__ recognize 47  : Recognized person with id 3 and distance 17.70268440246582
INFO       2024-04-17 18:55:52,036 __main__ recognize 47  : Recognized person with id 3 and distance 18.868776321411133
INFO       2024-04-17 18:55:52,277 __main__ recognize 47  : Recognized person with id 3 and distance 15.290412902832031
INFO       2024-04-17 18:55:52,524 __main__ recognize 47  : Recognized person with id 3 and distance 16.327228546142578
INFO       2024-04-17 18:55:52,785 __main__ recognize 47  : Recognized person with id 3 and distance 16.67988395690918
INFO       2024-04-17 18:55:53,037 __main__ recognize 47  : Recognized person with id 3 and distance 16.56026268005371
INFO       2024-04-17 18:55:53,278 __main__ recognize 47  : Recognized person with id 3 and distance 17.541839599609375
INFO       2024-04-17 18:55:53,520 __main__ recognize 47  : Recognized person with id 3 and distance 16.16450309753418
INFO       2024-04-17 18:55:53,766 __main__ recognize 47  : Recognized person with id 3 and distance 16.816471099853516
INFO       2024-04-17 18:55:54,010 __main__ recognize 47  : Recognized person with id 3 and distance 17.086708068847656
INFO       2024-04-17 18:55:54,255 __main__ recognize 47  : Recognized person with id 3 and distance 17.4269962310791
INFO       2024-04-17 18:55:54,497 __main__ recognize 47  : Recognized person with id 3 and distance 17.171049118041992
INFO       2024-04-17 18:55:54,738 __main__ recognize 47  : Recognized person with id 3 and distance 16.300912857055664
INFO       2024-04-17 18:55:54,981 __main__ recognize 47  : Recognized person with id 3 and distance 15.8547945022583
INFO       2024-04-17 18:55:55,240 __main__ recognize 47  : Recognized person with id 3 and distance 16.04002571105957
INFO       2024-04-17 18:55:55,488 __main__ recognize 47  : Recognized person with id 3 and distance 18.11513900756836
INFO       2024-04-17 18:55:55,736 __main__ recognize 47  : Recognized person with id 3 and distance 15.238873481750488
INFO       2024-04-17 18:55:55,997 __main__ recognize 47  : Recognized person with id 3 and distance 15.48730754852295
INFO       2024-04-17 18:55:56,257 __main__ recognize 47  : Recognized person with id 3 and distance 18.26038360595703
INFO       2024-04-17 18:55:56,502 __main__ recognize 47  : Recognized person with id 3 and distance 17.835031509399414
INFO       2024-04-18 09:26:37,905 pika.adapters.utils.io_services_utils abort 731 : Aborting transport connection: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 52950), raddr=('127.0.0.1', 5672)>
INFO       2024-04-18 09:26:37,905 pika.adapters.utils.io_services_utils _initiate_abort 904 : _AsyncTransportBase._initate_abort(): Initiating abrupt asynchronous transport shutdown: state=1; error=None; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 52950), raddr=('127.0.0.1', 5672)>
INFO       2024-04-18 09:26:37,905 pika.adapters.utils.io_services_utils _deactivate 869 : Deactivating transport: state=1; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 52950), raddr=('127.0.0.1', 5672)>
INFO       2024-04-18 09:26:37,905 pika.connection _on_stream_terminated 2017: AMQP stack terminated, failed to connect, or aborted: opened=True, error-arg=None; pending-error=ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
INFO       2024-04-18 09:26:37,905 pika.connection _on_stream_terminated 2086: Stack terminated due to ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
INFO       2024-04-18 09:26:37,905 pika.adapters.utils.io_services_utils _close_and_finalize 882 : Closing transport socket and unlinking: state=3; <socket.socket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 52950), raddr=('127.0.0.1', 5672)>
ERROR      2024-04-18 09:26:37,905 pika.adapters.blocking_connection _flush_output 521 : Unexpected connection close detected: ConnectionClosedByBroker: (320) "CONNECTION_FORCED - broker forced connection closure with reason 'shutdown'"
